--- make/pkgs/avahi/avahi.mk	2022-10-27 06:08:55.531234000 +0200
+++ make/pkgs/avahi/avahi.mk	2022-11-01 09:13:23.934171894 +0100
@@ -1,6 +1,6 @@
-$(call PKG_INIT_BIN, 0.6.31)
+$(call PKG_INIT_BIN, 0.6.32)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=8372719b24e2dd75de6f59bb1315e600db4fd092805bd1201ed0cb651a2dab48
+$(PKG)_HASH:=d54991185d514a0aba54ebeb408d7575b60f5818a772e28fa0e18b98bc1db454
 $(PKG)_SITE:=http://avahi.org/download
 
 $(PKG)_BINARIES := avahi-daemon $(if $(FREETZ_PACKAGE_AVAHI_WITH_DNSCONFD),avahi-dnsconfd)
--- make/pkgs/avahi/Config.in	2022-10-27 06:08:55.531234000 +0200
+++ make/pkgs/avahi/Config.in	2022-11-01 09:17:54.332413605 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_AVAHI
-	bool "Avahi 0.6.31"
+	bool "Avahi 0.6.32"
 	select FREETZ_LIB_libdl       if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libpthread  if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libexpat
--- make/pkgs/avahi/patches/110-nls_cleanups.patch	2022-10-27 06:08:55.531234000 +0200
+++ make/pkgs/avahi/patches/110-nls_cleanups.patch	2022-11-01 09:13:28.882061285 +0100
@@ -1,6 +1,6 @@
 --- configure
 +++ configure
-@@ -18784,6 +18784,7 @@
+@@ -19458,6 +19458,7 @@
  
  
  
@@ -8,7 +8,7 @@
  case "$am__api_version" in
      1.01234)
  	as_fn_error $? "Automake 1.5 or newer is required to use intltool" "$LINENO" 5
-@@ -20043,6 +20044,7 @@
+@@ -20700,6 +20701,7 @@
  
  
  avahilocaledir='${prefix}/${DATADIRNAME}/locale'
