--- make/pkgs/bvi/bvi.mk	2022-12-11 12:16:40.751585000 +0100
+++ make/pkgs/bvi/bvi.mk	2022-12-11 13:07:54.533880945 +0100
@@ -1,6 +1,6 @@
-$(call PKG_INIT_BIN, 1.4.0)
+$(call PKG_INIT_BIN, 1.4.1)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).src.tar.gz
-$(PKG)_HASH:=015a3c2832c7c097d98a5527deef882119546287ba8f2a70c736227d764ef802
+$(PKG)_HASH:=3035255ca79e0464567d255baa5544f7794e2b7eb791dcc60cc339cf1aa01e28
 $(PKG)_SITE:=@SF/bvi
 
 $(PKG)_DEPENDS_ON += ncurses
--- make/pkgs/bvi/Config.in	2022-12-11 12:16:40.751585000 +0100
+++ make/pkgs/bvi/Config.in	2022-12-11 13:09:10.312324717 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_BVI
-	bool "bvi/bmore 1.4.0 (binary only) "
+	bool "bvi/bmore 1.4.1 (binary only) "
 	default n
 	select FREETZ_LIB_libncurses
 	help
