--- make/pkgs/tiff/tiff.mk	2022-09-18 09:29:14.411122000 +0200
+++ make/pkgs/tiff/tiff.mk	2022-09-18 11:52:18.874578471 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, 4.0.7)
-$(PKG)_LIB_VERSION:=5.2.5
+$(call PKG_INIT_BIN, 4.5.0)
+$(PKG)_LIB_VERSION:=6.0.0
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=9f43a2cfb9589e5cecaa66e16bf87f814c945f22df7ba600d63aac4632c4f019
+$(PKG)_HASH:=c7a1d9296649233979fa3eacffef3fa024d73d05d589cb622727b5b08c423464
 $(PKG)_SITE:=http://download.osgeo.org/libtiff
 
 $(PKG)_BINARIES_ALL := fax2ps fax2tiff ppm2tiff raw2tiff tiff2bw tiff2pdf tiff2ps tiffinfo tiffsplit
--- make/pkgs/tiff/external.in.libs	2022-09-18 09:29:14.411122000 +0200
+++ make/pkgs/tiff/external.in.libs	2022-09-18 12:12:32.138004111 +0200
@@ -4,7 +4,7 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libtiff.so.5.2.5
+		 ${FREETZ_LIBRARY_DIR}/libtiff.so.6.0.0
 
 config EXTERNAL_FREETZ_LIB_libtiffxx
 	depends on EXTERNAL_ENABLED && FREETZ_LIB_libtiffxx
@@ -12,5 +12,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libtiffxx.so.5.2.5
+		 ${FREETZ_LIBRARY_DIR}/libtiffxx.so.6.0.0
 
--- make/pkgs/tiff/Config.in	2022-09-18 09:29:14.411122000 +0200
+++ make/pkgs/tiff/Config.in	2022-09-18 11:48:37.346982147 +0200
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_TIFF
-	bool "TIFF utils 4.0.7 (binary only)"
+	bool "TIFF utils 4.5.0 (binary only)"
 	default n
 	help
 		Various utilities to manupulate/convert images to/from TIFF format.
--- make/pkgs/tiff/patches/010-no_tests_no_docs.patch	2023-01-18 11:35:26.434569000 +0100
+++ make/pkgs/tiff/patches/010-no_tests_no_docs.patch	2023-01-18 11:42:38.556453160 +0100
@@ -1,15 +1,15 @@
 --- Makefile.in
 +++ Makefile.in
-@@ -436,7 +436,7 @@
- 	nmake.opt
+@@ -445,7 +445,7 @@
+ 	libtiff-4.pc.in
  
- dist_doc_DATA = $(docfiles)
--SUBDIRS = port libtiff tools build contrib test man html
+ @TIFF_DOCS_TRUE@dist_doc_DATA = $(docfiles)
+-SUBDIRS = port libtiff tools build contrib test doc
 +SUBDIRS = port libtiff tools build
  pkgconfigdir = $(libdir)/pkgconfig
  pkgconfig_DATA = libtiff-4.pc
  
-@@ -890,7 +890,7 @@
+@@ -916,7 +916,7 @@
  
  info-am:
  
--- make/pkgs/tiff/patches/020-ftell_build_fix.patch	2023-01-18 11:35:26.434569000 +0100
+++ make/pkgs/tiff/patches/020-ftell_build_fix.patch	2023-01-18 11:42:42.144404366 +0100
@@ -3,7 +3,7 @@
 
 --- libtiff/tiffiop.h
 +++ libtiff/tiffiop.h
-@@ -270,7 +270,7 @@
+@@ -315,7 +315,7 @@
    Windows read/write APIs support only 'unsigned int' rather than 'size_t'.
    Windows off_t is only 32-bit, even in 64-bit builds.
  */
