--- make/pkgs/tree/tree.mk	2022-10-27 06:08:55.631234000 +0200
+++ make/pkgs/tree/tree.mk	2022-11-05 23:54:54.684856513 +0100
@@ -1,7 +1,8 @@
-$(call PKG_INIT_BIN,1.8.0)
+$(call PKG_INIT_BIN,2.1.0)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tgz
-$(PKG)_HASH:=715d5d4b434321ce74706d0dd067505bb60c5ea83b5f0b3655dae40aa6f9b7c2
-$(PKG)_SITE:=ftp://mama.indstate.edu/linux/tree
+$(PKG)_HASH:=0160c535bff2b0dc6a830b9944e981e3427380f63e748da96ced7071faebabf6
+#$(PKG)_SITE:=ftp://mama.indstate.edu/linux/tree
+$(PKG)_SITE:=https://mama.indstate.edu/users/ice/tree/src/
 $(PKG)_BINARY:=$($(PKG)_DIR)/tree
 $(PKG)_TARGET_BINARY:=$($(PKG)_DEST_DIR)/usr/bin/tree
 
--- make/pkgs/tree/Config.in	2022-10-27 06:08:55.631234000 +0200
+++ make/pkgs/tree/Config.in	2022-11-06 08:48:55.636781174 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_TREE
-	bool "tree 1.8.0 (binary only)"
+	bool "tree 2.1.0 (binary only)"
 	default n
 	help
 		Tree is a recursive directory listing program
--- make/pkgs/tree/patches/100-makefile.patch	2022-11-06 08:45:13.314864575 +0100
+++ make/pkgs/tree/patches/100-makefile.patch	2022-11-06 08:45:24.086729261 +0100
@@ -1,12 +1,12 @@
 --- Makefile
 +++ Makefile
-@@ -32,7 +32,8 @@
+@@ -34,7 +34,8 @@
  # Linux defaults:
- CFLAGS=-ggdb -pedantic -Wall -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
- #CFLAGS=-O4 -Wall  -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
--#LDFLAGS=-s
+ #CFLAGS+=-ggdb -std=c11 -pedantic -Wall -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+ CFLAGS+=-O3 -std=c11 -pedantic -Wall -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+-#LDFLAGS+=-s
 +LDFLAGS=
 +OBJS+=strverscmp.o
  
  # Uncomment for FreeBSD:
- #CFLAGS=-O2 -Wall -fomit-frame-pointer
+ #CC=cc
