--- make/host-tools/sed-host/sed-host.mk	2023-02-21 08:37:44.003217269 +0100
+++ make/host-tools/sed-host/sed-host.mk	2023-02-21 03:48:29.934464000 +0100
@@ -1,6 +1,6 @@
-$(call TOOLS_INIT, 4.9)
+$(call TOOLS_INIT, 4.8)
 $(PKG)_SOURCE:=sed-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=6e226b732e1cd739464ad6862bd1a1aba42d7982922da7a53519631d24975181
+$(PKG)_HASH:=f79b0cfea71b37a8eeec8490db6c5f7ae7719c35587f21edb0617f370eeff633
 $(PKG)_SITE:=@GNU/sed
 
 $(PKG)_CONFIGURE_OPTIONS += --prefix=/usr
