--- config/avm/availability.in
+++ config/avm/availability.in	2024-06-12 17:30:39.489487086 +0200
@@ -415,8 +415,18 @@
 	depends on \
 		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
 			FREETZ_TYPE_4050 || \
-			FREETZ_TYPE_5690 \
-		))
+			FREETZ_TYPE_5690 || \
+			FREETZ_TYPE_7510 || \
+			FREETZ_TYPE_7520 || \
+			FREETZ_TYPE_7521 || \
+			FREETZ_TYPE_7590 || \
+			FREETZ_TYPE_7690 || \
+			FREETZ_TYPE_GATE \
+	)) \
+		|| \
+		( FREETZ_TYPE_LANG_DE && ( \
+			FREETZ_TYPE_6670 \
+ 		))
 
 config FREETZ_AVM_HAS_FIRMWARE_07_5X
 	bool
@@ -467,6 +477,14 @@
 			FREETZ_TYPE_6820_V3 \
 		))
 
+config FREETZ_AVM_HAS_FIRMWARE_07_5X_PLUS
+ 	bool
+ 	default y
+	depends on \
+	 	( (FREETZ_TYPE_LANG_DE) && ( \
+			FREETZ_TYPE_6660 || \
+			FREETZ_TYPE_6690 \
+ 		))
 
 config FREETZ_AVM_HAS_FIRMWARE_07_8X
 	bool
@@ -479,14 +497,35 @@
 			FREETZ_TYPE_7599 \
 		))
 
+config FREETZ_AVM_HAS_FIRMWARE_07_8X_INHAUS
+	bool
+	default y
+	depends on \
+		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
+			FREETZ_TYPE_5530 || \
+			FREETZ_TYPE_5590 || \
+			FREETZ_TYPE_7599 \
+		))
 
 config FREETZ_AVM_HAS_FIRMWARE_08_0X_INHAUS
 	bool
 	default y
 	depends on \
 		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
+			FREETZ_TYPE_4060 || \
+			FREETZ_TYPE_5530 || \
+			FREETZ_TYPE_5590 || \
 			FREETZ_TYPE_6850_4G || \
-			FREETZ_TYPE_6850_5G \
+			FREETZ_TYPE_6850_5G || \
+			FREETZ_TYPE_7539 || \
+			FREETZ_TYPE_7590 || \
+			FREETZ_TYPE_7599 \
+		)) \
+		|| \
+		( FREETZ_TYPE_LANG_DE && ( \
+			FREETZ_TYPE_6591 || \
+			FREETZ_TYPE_6660 || \
+			FREETZ_TYPE_6690 \
 		))
 
 config FREETZ_AVM_HAS_FIRMWARE_08_0X_LABOR
