--- config/avm/availability.in
+++ config/avm/availability.in	2024-08-07 17:13:45.727947068 +0200
@@ -406,9 +406,25 @@
 	default y
 	depends on \
 		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
-			FREETZ_TYPE_5690 \
+			FREETZ_TYPE_5690 || \
+			FREETZ_TYPE_GATE \
+		)) \
+		|| \
+		( FREETZ_TYPE_LANG_DE && ( \
+			FREETZ_TYPE_6660 || \
+			FREETZ_TYPE_6690 \
 		))
 
+config FREETZ_AVM_HAS_FIRMWARE_07_5X_INHAUS
+	bool
+	default y
+	depends on \
+		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
+			FREETZ_TYPE_5690 || \
+			FREETZ_TYPE_7690 || \
+			FREETZ_TYPE_GATE \
+	))
+
 config FREETZ_AVM_HAS_FIRMWARE_07_5X
 	bool
 	default y
@@ -476,10 +492,25 @@
 	default y
 	depends on \
 		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
+			FREETZ_TYPE_3009 || \
 			FREETZ_TYPE_4040 || \
 			FREETZ_TYPE_4050 || \
+			FREETZ_TYPE_4060 || \
+			FREETZ_TYPE_5530 || \
+			FREETZ_TYPE_5590 || \
 			FREETZ_TYPE_6850_4G || \
-			FREETZ_TYPE_6850_5G \
+			FREETZ_TYPE_6850_5G || \
+			FREETZ_TYPE_7530 || \
+			FREETZ_TYPE_7539 || \
+			FREETZ_TYPE_7590 || \
+			FREETZ_TYPE_7599 \
+		)) \
+		|| \
+		( FREETZ_TYPE_LANG_DE && ( \
+			FREETZ_TYPE_6591 || \
+			FREETZ_TYPE_6660 || \
+			FREETZ_TYPE_6670 || \
+			FREETZ_TYPE_6690 \
 		))
 
 config FREETZ_AVM_HAS_FIRMWARE_08_0X_LABOR
