--- config/avm/availability.in
+++ config/avm/availability.in	2024-09-17 17:51:31.134393357 +0200
@@ -400,7 +400,6 @@
 			FREETZ_TYPE_6820_V2 \
 		))
 
-
 config FREETZ_AVM_HAS_FIRMWARE_07_5X
 	bool
 	default y
@@ -450,6 +449,22 @@
 			FREETZ_TYPE_6690 || \
 			FREETZ_TYPE_6820_V3 \
 		))
+		
+config FREETZ_AVM_HAS_FIRMWARE_07_5X_LABOR
+	bool
+	default y
+	depends on \
+		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
+			FREETZ_TYPE_GATE \
+		)) 
+
+config FREETZ_AVM_HAS_FIRMWARE_07_5X_INHAUS
+	bool
+	default y
+	depends on \
+		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
+			FREETZ_TYPE_GATE \
+	))
 
 
 config FREETZ_AVM_HAS_FIRMWARE_07_8X
@@ -470,9 +485,24 @@
 	depends on \
 		( (FREETZ_TYPE_LANG_DE||FREETZ_TYPE_LANG_EN) && ( \
 			FREETZ_TYPE_4050 || \
+			FREETZ_TYPE_4060 || \
+			FREETZ_TYPE_5530 || \
+			FREETZ_TYPE_5590 || \
 			FREETZ_TYPE_5690_PRO || \
 			FREETZ_TYPE_6850_4G || \
-			FREETZ_TYPE_6850_5G \
+			FREETZ_TYPE_6850_5G || \
+			FREETZ_TYPE_7530 || \
+			FREETZ_TYPE_7539 || \
+			FREETZ_TYPE_7590 || \
+			FREETZ_TYPE_7599 || \
+			FREETZ_TYPE_7690 \
+		)) \
+		|| \
+		( FREETZ_TYPE_LANG_DE && ( \
+			FREETZ_TYPE_6591 || \
+			FREETZ_TYPE_6660 || \
+			FREETZ_TYPE_6670 || \
+			FREETZ_TYPE_6690 \
 		))
 
 config FREETZ_AVM_HAS_FIRMWARE_08_0X_LABOR
@@ -484,6 +514,8 @@
 			FREETZ_TYPE_4060 || \
 			FREETZ_TYPE_5530 || \
 			FREETZ_TYPE_5590 || \
+			FREETZ_TYPE_6850_4G || \
+			FREETZ_TYPE_6850_5G || \
 			FREETZ_TYPE_7530_W5 || \
 			FREETZ_TYPE_7530_W6 || \
 			FREETZ_TYPE_7590 || \
