--- make/host-tools/yf-bootmanager-host/yf-bootmanager-host.mk	2022-11-24 20:17:20.133249000 +0100
+++ make/host-tools/yf-bootmanager-host/yf-bootmanager-host.mk	2022-12-02 22:30:30.573064552 +0100
@@ -1,5 +1,5 @@
 YF_BOOTMANAGER_HOST_REPOSITORY:=https://github.com/PeterPawn/YourFritz.git
-$(call TOOLS_INIT, $(if $(FREETZ_PATCH_MODFS_BOOT_MANAGER_labor),$(call git-get-tag-revision,$(YF_BOOTMANAGER_HOST_REPOSITORY),freetz-ng-test),$(if $(FREETZ_PATCH_MODFS_BOOT_MANAGER_latest),$(call git-get-tag-revision,$(YF_BOOTMANAGER_HOST_REPOSITORY),freetz-ng-version),87ee8c3b73edf4ba70025a017aa1d49cd2c66036)))
+$(call TOOLS_INIT, $(if $(FREETZ_PATCH_MODFS_BOOT_MANAGER_labor),$(call git-get-tag-revision,$(YF_BOOTMANAGER_HOST_REPOSITORY),freetz-ng-test),$(if $(FREETZ_PATCH_MODFS_BOOT_MANAGER_latest),$(call git-get-tag-revision,$(YF_BOOTMANAGER_HOST_REPOSITORY),freetz-ng-version),639a7202a7d9b4cdc6d1c3b425c0a25759c5e0d8)))
 # Versions after this commit have no vanilla GPL3 - but it's not a problem to use the unchanged(!) version in an own image.
 $(PKG)_SOURCE:=yf-bootmanager-$($(PKG)_VERSION).tar.xz
 $(PKG)_HASH:=f47e9395048d2d095f8d6a5286941adf9da837bb303b2366466365b2f984c1bb
