--- make/mod/files/root/etc/default.mod/motd	2022-06-27 20:48:34.817796000 +0200
+++ make/mod/files/root/etc/default.mod/motd	2022-06-27 21:01:47.289982749 +0200
@@ -8,5 +8,18 @@
   | |_ | '__/ _ \/ _ \ __|_  /____|  \| | |  _
   |  _|| | |  __/  __/ |_ / /_____| |\  | |_| |
   |_|  |_|  \___|\___|\__/___|    |_| \_|\____|
+                  _           
+                 | |__  _   _ 
+                 | '_ \| | | |
+                 | |_) | |_| |
+                 |_.__/ \__, |
+                        |___/ 
+      ___      __  __    _  _____ ___ _  __
+     |_ _|    |  \/  |  / \|_   _|_ _| |/ /
+      | |_____| |\/| | / _ \ | |  | || ' / 
+      | |_____| |  | |/ ___ \| |  | || . \ 
+     |___|    |_|  |_/_/   \_\_| |___|_|\_\\
+     https://github.com/MasterRoCcO/i-matik
+                  Version xxx
 EOX
 
--- make/mod/files/root/usr/lib/mww/cgi/menu-new.sh	2022-06-13 18:38:11.470319488 +0200
+++ make/mod/files/root/usr/lib/mww/cgi/menu-new.sh	2022-06-13 18:46:28.568103025 +0200
@@ -12,7 +12,7 @@
 	local id=$1
 	case $id in
 		status*|daemons) sub=status ;;
-		system|avmwif_*|rudi_*|firmware_*|backup_*|support_*|freetz) sub=system ;;
+		system|avmwif_*|rudi_*|firmware_*|backup_*|support_*|freetz|i-matik) sub=system ;;
 		*:*) sub=${id#*:}; sub="pkg:${sub%%:*}" ;;
 	esac
 	echo "$sub"
@@ -198,6 +198,7 @@
 <li><a id="rudi_shell" href="/cgi-bin/shell/index.cgi" target="_blank">$(lang de:"Rudi-Shell" en:"Rudi shell")</a></li>
 <li><a id="avmwif_link" href="/cgi-bin/avm" target="_blank">$(lang de:"AVM-Webinterface" en:"AVM web interface")</a></li>
 <li><a id="freetz" href="/cgi-bin/freetz.cgi">$(lang de:"&Uuml;ber" en:"About") Freetz</a></li>
+<li><a id="i-matik" href="/cgi-bin/i-matik.cgi">$(lang de:"Gebaut mit" en:"Built with") I-matik</a></li>
 EOF
 	} > "$dir/system.sub"
 
--- make/mod/files/root/usr/lib/cgi-bin/mod/info.cgi	2022-09-22 19:57:32.363196000 +0200
+++ make/mod/files/root/usr/lib/cgi-bin/mod/info.cgi	2022-09-24 18:17:15.751284126 +0200
@@ -25,6 +25,7 @@
 date_de_format=$(echo "$FREETZ_INFO_MAKEDATE" \
 		| sed -re 's/([0-9]{4})([0-9]{2})([0-9]{2})-([0-9]{2})([0-9]{2})([0-9]{2})(.*)/\3.\2.\1 \4\:\5\:\6/')
 echo "<dl class='info'><dt>$(lang de:"Erstellungsdatum" en:"Creation date")</dt><dd>$date_de_format</dd></dl>"
+echo "<dl class='info'><dt>$(lang de:"i-matik version" en:"i-matik version")</dt><dd>xxxx</dd></dl>"
 echo "<dl class='info'><dt>$(lang de:"Urspr&uuml;nglicher Dateiname" en:"Initial file name")</dt><dd>$FREETZ_INFO_IMAGE_NAME</dd></dl>"
 if [ ! -z "$FREETZ_INFO_COMMENT" ]; then
 	echo "<dl class='info'><dt>$(lang de:"Benutzerdefinierte Informationen" en:"User defined information")</dt><dd>$FREETZ_INFO_COMMENT</dd></dl>"
