--- make/mod/files/root/usr/lib/mww/cgi/menu-new.sh	2022-06-11 23:45:54.694383355 +0200
+++ make/mod/files/root/usr/lib/mww/cgi/menu-new.sh	2022-06-11 23:48:24.807388188 +0200
@@ -12,7 +12,7 @@
 	local id=$1
 	case $id in
 		status*|daemons) sub=status ;;
-		system|avmwif_*|rudi_*|firmware_*|backup_*|support_*|freetz) sub=system ;;
+		system|avmwif_*|rudi_*|firmware_*|backup_*|support_*|freetz|i-matik) sub=system ;;
 		*:*) sub=${id#*:}; sub="pkg:${sub%%:*}" ;;
 	esac
 	echo "$sub"
@@ -193,6 +193,7 @@
 <li><a id="rudi_shell" href="/cgi-bin/shell/index.cgi" target="_blank">$(lang de:"Rudi-Shell" en:"Rudi shell")</a></li>
 <li><a id="avmwif_link" href="/cgi-bin/avm" target="_blank">$(lang de:"AVM-Webinterface" en:"AVM web interface")</a></li>
 <li><a id="freetz" href="/cgi-bin/freetz.cgi">$(lang de:"&Uuml;ber" en:"About") Freetz</a></li>
+<li><a id="i-matik" href="/cgi-bin/i-matik.cgi">$(lang de:"Gebaut mit" en:"Built with") I-matik</a></li>
 EOF
 	} > "$dir/system.sub"
 
