--- make/pcsc-lite/Config.in	2022-08-17 10:14:10.087988000 +0200
+++ make/pcsc-lite/Config.in	2022-09-14 20:03:09.158580565 +0200
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_PCSC_LITE
-	bool "PCSC-lite 1.9.8 (binary only)"
+	bool "PCSC-lite 1.9.9 (binary only)"
 	select FREETZ_LIB_libdl       if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libpthread  if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_librt       if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
--- make/pcsc-lite/pcsc-lite.mk	2022-08-17 10:14:10.087988000 +0200
+++ make/pcsc-lite/pcsc-lite.mk	2022-09-14 20:06:23.934252898 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, 1.9.8)
+$(call PKG_INIT_BIN, 1.9.9)
 $(PKG)_LIB_VERSION:=1.0.0
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.bz2
-$(PKG)_HASH:=502d80c557ecbee285eb99fe8703eeb667bcfe067577467b50efe3420d1b2289
+$(PKG)_HASH:=cbcc3b34c61f53291cecc0d831423c94d437b188eb2b97b7febc08de1c914e8a
 $(PKG)_SITE:=https://pcsclite.apdu.fr/files
 ### WEBSITE:=https://pcsclite.apdu.fr/
 ### MANPAGE:=https://salsa.debian.org/rousseau/PCSC/blob/master/README
--- make/cifsmount/cifsmount.mk	2022-09-17 20:13:16.957299643 +0200
+++ make/cifsmount/cifsmount.mk	2022-09-17 20:12:06.938753664 +0200
@@ -1,6 +1,6 @@
-$(call PKG_INIT_BIN, 6.15)
+$(call PKG_INIT_BIN, 7.0)
 $(PKG)_SOURCE:=cifs-utils-$($(PKG)_VERSION).tar.bz2
-$(PKG)_HASH:=a7b6940e93250c1676a6fa66b6ead91b78cd43a5fee99cc462459c8b9cf1e6f4
+$(PKG)_HASH:=0defaab85bd3ea46ffc45ab41fb0d0ad54d05ae2cfaa7e503de86d4f12bc8161
 $(PKG)_SITE:=@SAMBA/linux-cifs/cifs-utils
 ### WEBSITE:=https://wiki.samba.org/index.php/LinuxCIFS_utils
 ### CHANGES:=https://wiki.samba.org/index.php/LinuxCIFS_utils#News
--- make/cifsmount/Config.in	2022-08-17 10:14:09.959990000 +0200
+++ make/cifsmount/Config.in	2022-09-17 20:08:51.094804384 +0200
@@ -2,7 +2,7 @@
 	depends on !(FREETZ_AVM_HAS_CIFS_BUILTIN || FREETZ_REPLACE_MODULE_AVAILABLE)
 
 config FREETZ_PACKAGE_CIFSMOUNT
-	bool "Cifsmount 6.15"
+	bool "Cifsmount 7.0"
 	depends on FREETZ_AVM_HAS_CIFS_BUILTIN || FREETZ_REPLACE_MODULE_AVAILABLE
 	select FREETZ_MODULE_cifs if !FREETZ_AVM_HAS_CIFS_BUILTIN
 	default n
--- make/imagemagick/Config.in	2022-09-18 09:29:13.343138420 +0200
+++ make/imagemagick/Config.in	2022-09-18 10:21:03.317815259 +0200
@@ -1,5 +1,5 @@
 menuconfig FREETZ_PACKAGE_IMAGEMAGICK
-	bool "ImageMagick 7.0.10-10 (binary only)"
+	bool "ImageMagick 7.1.0-48 (binary only)"
 	select FREETZ_LIB_libpthread  if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libm        if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	default n
@@ -39,7 +39,7 @@
 config FREETZ_PACKAGE_IMAGEMAGICK_png
 	bool "Support png"
 	default y
-	select FREETZ_LIB_libpng12
+	select FREETZ_LIB_libpng16
 
 config FREETZ_PACKAGE_IMAGEMAGICK_xml
 	bool "Support xml"
--- make/imagemagick/external.files	2022-09-18 09:29:13.343138000 +0200
+++ make/imagemagick/external.files	2022-09-18 10:19:37.523708711 +0200
@@ -1,6 +1,6 @@
 if [ "$EXTERNAL_FREETZ_PACKAGE_IMAGEMAGICK" ]; then
 	[ "$FREETZ_PACKAGE_IMAGEMAGICK"        == "y" ] && EXTERNAL_FILES+=" /usr/bin/magick"
-	[ "$FREETZ_PACKAGE_IMAGEMAGICK_STATIC" != "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libMagickCore-7-Q16.so.7.0.0"
-	[ "$FREETZ_PACKAGE_IMAGEMAGICK_STATIC" != "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libMagickWand-7-Q16.so.7.0.0"
+	[ "$FREETZ_PACKAGE_IMAGEMAGICK_STATIC" != "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libMagickCore-7-Q16.so.10"
+	[ "$FREETZ_PACKAGE_IMAGEMAGICK_STATIC" != "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libMagickWand-7-Q16.so.10"
 	[ "$FREETZ_PACKAGE_IMAGEMAGICK_xml"    == "y" ] && EXTERNAL_FILES+=" /etc/ImageMagick-7"
 fi
--- make/imagemagick/external.in	2022-09-18 09:29:13.343138000 +0200
+++ make/imagemagick/external.in	2022-09-18 10:19:58.095248620 +0200
@@ -5,7 +5,7 @@
 	help
 		externals the following file(s):
 		 /usr/bin/magick
-		 ${FREETZ_LIBRARY_DIR}/libMagickWand-7-Q16.so.7.0.0
-		 ${FREETZ_LIBRARY_DIR}/libMagickCore-7-Q16.so.7.0.0
+		 ${FREETZ_LIBRARY_DIR}/libMagickWand-7-Q16.so.10
+		 ${FREETZ_LIBRARY_DIR}/libMagickCore-7-Q16.so.10
 		 /etc/ImageMagick-7
 
--- make/imagemagick/imagemagick.mk	2022-09-18 09:29:13.343138000 +0200
+++ make/imagemagick/imagemagick.mk	2022-09-18 10:17:42.090371767 +0200
@@ -1,9 +1,9 @@
-$(call PKG_INIT_BIN, 7.0.10-10)
+$(call PKG_INIT_BIN, 7.1.0-48)
 $(PKG)_MAJOR_VERSION:=$(call GET_MAJOR_VERSION,$($(PKG)_VERSION),1)
 $(PKG)_ABI_SUFFIX:=Q16
 $(PKG)_SOURCE:=ImageMagick-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=df1a37b73aa49423abb422c2150a0e1436920ba50dfc4377c6a3793f9826e5f1
-$(PKG)_SITE:=@SF/$(pkg),http://www.imagemagick.org/download
+$(PKG)_HASH:=eb2d4dc3cc9fd3d4686aa64562177ca5067b1d40bcb6ec75fa1003eaf8c37f4e
+$(PKG)_SITE:=@SF/$(pkg),https://mirror.dogado.de/imagemagick,http://www.imagemagick.org/download
 
 $(PKG)_BINARY := magick
 $(PKG)_BINARY_BUILD_DIR := $($(PKG)_BINARY:%=$($(PKG)_DIR)/utilities/$(if $(FREETZ_PACKAGE_IMAGEMAGICK_STATIC),,.libs/)%)
@@ -12,11 +12,11 @@
 $(PKG)_SYMLINKS := animate compare composite conjure convert display identify import magick-script mogrify montage stream
 $(PKG)_SYMLINKS_TARGET_DIR := $($(PKG)_SYMLINKS:%=$($(PKG)_DEST_DIR)/usr/bin/%)
 
-$(PKG)_LIB_CORE := libMagickCore-$($(PKG)_MAJOR_VERSION)-$($(PKG)_ABI_SUFFIX).so.7.0.0
+$(PKG)_LIB_CORE := libMagickCore-$($(PKG)_MAJOR_VERSION)-$($(PKG)_ABI_SUFFIX).so.10
 $(PKG)_LIB_CORE_BUILD_DIR := $($(PKG)_LIB_CORE:%=$($(PKG)_DIR)/MagickCore/.libs/%)
 $(PKG)_LIB_CORE_TARGET_DIR := $($(PKG)_LIB_CORE:%=$($(PKG)_DEST_LIBDIR)/%)
 
-$(PKG)_LIB_WAND := libMagickWand-$($(PKG)_MAJOR_VERSION)-$($(PKG)_ABI_SUFFIX).so.7.0.0
+$(PKG)_LIB_WAND := libMagickWand-$($(PKG)_MAJOR_VERSION)-$($(PKG)_ABI_SUFFIX).so.10
 $(PKG)_LIB_WAND_BUILD_DIR := $($(PKG)_LIB_WAND:%=$($(PKG)_DIR)/MagickWand/.libs/%)
 $(PKG)_LIB_WAND_TARGET_DIR := $($(PKG)_LIB_WAND:%=$($(PKG)_DEST_LIBDIR)/%)
 
--- make/libs/leptonica/leptonica.mk	2022-09-18 09:29:13.503136000 +0200
+++ make/libs/leptonica/leptonica.mk	2022-09-18 11:07:30.720885006 +0200
@@ -1,8 +1,8 @@
-$(call PKG_INIT_LIB, 1.69)
-$(PKG)_LIB_VERSION:=3.0.0
-$(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.bz2
-$(PKG)_HASH:=e8a4e327f0402073c967bf2da976dc914b9086b6bd2b48558307c928a0960b6d
-$(PKG)_SITE:=http://leptonica.googlecode.com/files
+$(call PKG_INIT_LIB, 1.82.0)
+$(PKG)_LIB_VERSION:=5
+$(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
+$(PKG)_HASH:=155302ee914668c27b6fe3ca9ff2da63b245f6d62f3061c8f27563774b8ae2d6
+$(PKG)_SITE:=https://github.com/DanBloomberg/leptonica/releases/download/$($(PKG)_VERSION)
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/src/.libs/liblept.so.$($(PKG)_LIB_VERSION)
 $(PKG)_STAGING_BINARY:=$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/liblept.so.$($(PKG)_LIB_VERSION)
--- make/libs/leptonica/patches/010-enforce-libpng12.leptonica.patch	2022-09-18 09:29:13.507135000 +0200
+++ make/libs/leptonica/patches/010-enforce-libpng12.leptonica.patch	2022-09-18 11:13:03.342545193 +0200
@@ -1,28 +1,11 @@
 --- configure
 +++ configure
-@@ -11402,13 +11402,13 @@
- fi
+@@ -13603,7 +13603,7 @@
  
- if test "x$with_libpng" != xno; then :
--  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for png_read_png in -lpng" >&5
--$as_echo_n "checking for png_read_png in -lpng... " >&6; }
-+  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for png_read_png in -lpng12" >&5
-+$as_echo_n "checking for png_read_png in -lpng12... " >&6; }
- if test "${ac_cv_lib_png_png_read_png+set}" = set; then :
-   $as_echo_n "(cached) " >&6
- else
-   ac_check_lib_save_LIBS=$LIBS
--LIBS="-lpng  $LIBS"
-+LIBS="-lpng12  $LIBS"
- cat confdefs.h - <<_ACEOF >conftest.$ac_ext
- /* end confdefs.h.  */
- 
-@@ -11441,7 +11441,7 @@
- if test "x$ac_cv_lib_png_png_read_png" = x""yes; then :
  
  $as_echo "#define HAVE_LIBPNG 1" >>confdefs.h
 - LIBPNG_LIBS=-lpng
-+ LIBPNG_LIBS=-lpng12
++ LIBPNG_LIBS=-lpng16
+ 
  
  else
-   if test "x$with_libpng" = xyes; then :
--- make/libs/leptonica/Config.in	2022-08-17 09:18:28.378826000 +0200
+++ make/libs/leptonica/Config.in	2022-09-17 18:12:53.845947247 +0200
@@ -2,7 +2,7 @@
 	bool "leptonica (liblept.so)"
 	select FREETZ_LIB_libm if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libjpeg
-	select FREETZ_LIB_libpng12
+	select FREETZ_LIB_libpng16
 	select FREETZ_LIB_libtiff
 	select FREETZ_LIB_libz
 	default n
--- make/libs/leptonica/external.files	2022-09-18 09:29:13.503136000 +0200
+++ make/libs/leptonica/external.files	2022-09-18 11:30:16.908848605 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_liblept" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/liblept.so.3.0.0"
+[ "$EXTERNAL_FREETZ_LIB_liblept" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/liblept.so.5"
--- make/libs/leptonica/external.in	2022-09-18 09:29:13.503136000 +0200
+++ make/libs/leptonica/external.in	2022-09-18 11:30:30.696566918 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/liblept.so.3.0.0
+		 ${FREETZ_LIBRARY_DIR}/liblept.so.5
 
--- make/tiff/tiff.mk	2022-09-18 09:29:14.411122000 +0200
+++ make/tiff/tiff.mk	2022-09-18 11:52:18.874578471 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, 4.0.7)
-$(PKG)_LIB_VERSION:=5.2.5
+$(call PKG_INIT_BIN, 4.4.0)
+$(PKG)_LIB_VERSION:=5
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=9f43a2cfb9589e5cecaa66e16bf87f814c945f22df7ba600d63aac4632c4f019
+$(PKG)_HASH:=917223b37538959aca3b790d2d73aa6e626b688e02dcda272aec24c2f498abed
 $(PKG)_SITE:=http://download.osgeo.org/libtiff
 
 $(PKG)_BINARIES_ALL := fax2ps fax2tiff ppm2tiff raw2tiff tiff2bw tiff2pdf tiff2ps tiffinfo tiffsplit
--- make/tiff/external.in.libs	2022-09-18 09:29:14.411122000 +0200
+++ make/tiff/external.in.libs	2022-09-18 12:12:32.138004111 +0200
@@ -4,7 +4,7 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libtiff.so.5.2.5
+		 ${FREETZ_LIBRARY_DIR}/libtiff.so.5
 
 config EXTERNAL_FREETZ_LIB_libtiffxx
 	depends on EXTERNAL_ENABLED && FREETZ_LIB_libtiffxx
@@ -12,5 +12,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libtiffxx.so.5.2.5
+		 ${FREETZ_LIBRARY_DIR}/libtiffxx.so.5
 
--- make/tiff/Config.in	2022-09-18 09:29:14.411122000 +0200
+++ make/tiff/Config.in	2022-09-18 11:48:37.346982147 +0200
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_TIFF
-	bool "TIFF utils 4.0.7 (binary only)"
+	bool "TIFF utils 4.4.0 (binary only)"
 	default n
 	help
 		Various utilities to manupulate/convert images to/from TIFF format.
--- make/libs/libxml2/libxml2.mk	2022-09-18 13:06:37.109155000 +0200
+++ make/libs/libxml2/libxml2.mk	2022-09-20 16:52:21.627075328 +0200
@@ -1,8 +1,8 @@
-$(call PKG_INIT_LIB, 2.9.14)
+$(call PKG_INIT_LIB, 2.10.2)
 $(PKG)_LIB_VERSION:=$($(PKG)_VERSION)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=60d74a257d1ccec0475e749cba2f21559e48139efba6ff28224357c7c798dfee
-$(PKG)_SITE:=https://download.gnome.org/sources/libxml2/2.9
+$(PKG)_HASH:=d240abe6da9c65cb1900dd9bf3a3501ccf88b3c2a1cb98317d03f272dda5b265
+$(PKG)_SITE:=https://download.gnome.org/sources/libxml2/2.10
 ### WEBSITE:=http://www.xmlsoft.org
 ### MANPAGE:=https://gitlab.gnome.org/GNOME/libxml2/-/wikis/home#html-documentation
 ### CHANGES:=https://gitlab.gnome.org/GNOME/libxml2/-/releases
--- make/libs/libxml2/external.in	2022-09-18 13:06:37.109155000 +0200
+++ make/libs/libxml2/external.in	2022-09-20 18:01:53.411597559 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libxml2.so.2.9.14
+		 ${FREETZ_LIBRARY_DIR}/libxml2.so.2.10.2
 
--- make/libs/libxml2/external.files	2022-09-18 13:06:37.109155000 +0200
+++ make/libs/libxml2/external.files	2022-09-20 18:01:03.336585336 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libxml2" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libxml2.so.2.9.14"
+[ "$EXTERNAL_FREETZ_LIB_libxml2" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libxml2.so.2.10.2"
--- make/libs/libxml2/patches/100-no_docs_examples_tests.libxml2.patch	2022-09-20 18:34:32.297662535 +0200
+++ make/libs/libxml2/patches/100-no_docs_examples_tests.libxml2.patch	2022-09-20 18:37:23.150133937 +0200
@@ -1,49 +1,47 @@
---- Makefile.in
-+++ Makefile.in
-@@ -411,7 +411,7 @@
- man3dir = $(mandir)/man3
- NROFF = nroff
- MANS = $(man_MANS)
--DATA = $(cmake_DATA) $(confexec_DATA) $(m4data_DATA) $(pkgconfig_DATA)
-+DATA =                                               $(pkgconfig_DATA)
+--- Makefile.in	2022-08-29 15:20:52.000000000 +0200
++++ Makefile.in	2022-09-20 17:55:12.299851931 +0200
+@@ -373,7 +373,7 @@
+     n|no|NO) false;; \
+     *) (install-info --version) >/dev/null 2>&1;; \
+   esac
+-DATA = $(cmake_DATA) $(examples_DATA) $(m4data_DATA) \
++DATA = $(cmake_DATA) \
+ 	$(nodist_pkgconfig_DATA)
  RECURSIVE_CLEAN_TARGETS = mostlyclean-recursive clean-recursive	\
    distclean-recursive maintainer-clean-recursive
- am__recursive_targets = \
-@@ -724,7 +724,7 @@
+@@ -677,7 +677,7 @@
  top_builddir = @top_builddir@
  top_srcdir = @top_srcdir@
  ACLOCAL_AMFLAGS = -I m4
--SUBDIRS = include . doc example fuzz xstc $(PYTHON_SUBDIR)
-+SUBDIRS = include .                  xstc $(PYTHON_SUBDIR)
+-SUBDIRS = include . doc example fuzz xstc $(am__append_1)
++SUBDIRS = include .                  xstc $(am__append_1)
  DIST_SUBDIRS = include . doc example fuzz python xstc
- AM_CPPFLAGS = -I$(top_builddir)/include -I$(srcdir)/include
- AM_CFLAGS = $(EXTRA_CFLAGS) $(THREAD_CFLAGS) $(Z_CFLAGS) $(LZMA_CFLAGS)
-@@ -1769,7 +1769,7 @@
+ AM_CPPFLAGS = -I$(top_builddir)/include -I$(srcdir)/include -DSYSCONFDIR='"$(sysconfdir)"'
+ AM_CFLAGS = $(EXTRA_CFLAGS)
+@@ -1906,7 +1906,7 @@
  	$(MAKE) $(AM_MAKEFLAGS) $(check_PROGRAMS)
- check: $(BUILT_SOURCES)
- 	$(MAKE) $(AM_MAKEFLAGS) check-recursive
--all-am: Makefile $(PROGRAMS) $(LTLIBRARIES) $(SCRIPTS) $(MANS) $(DATA) \
-+all-am: Makefile             $(LTLIBRARIES)            $(MANS) $(DATA) \
+ 	$(MAKE) $(AM_MAKEFLAGS) check-local
+ check: check-recursive
+-all-am: Makefile $(PROGRAMS) $(LTLIBRARIES) $(SCRIPTS) $(DATA) \
++all-am: Makefile             $(LTLIBRARIES)            $(DATA) \
  		config.h
  install-binPROGRAMS: install-libLTLIBRARIES
  
-@@ -1906,15 +1906,15 @@
+@@ -2033,14 +2033,14 @@
  
  info-am:
  
--install-data-am: install-cmakeDATA install-data-local \
--	install-m4dataDATA install-man install-pkgconfigDATA
-+install-data-am: \
-+	install-pkgconfigDATA
+-install-data-am: install-cmakeDATA install-examplesDATA \
+-	install-m4dataDATA install-nodist_pkgconfigDATA
++install-data-am:  \
++	install-nodist_pkgconfigDATA
  
  install-dvi: install-dvi-recursive
  
  install-dvi-am:
  
--install-exec-am: install-binPROGRAMS install-binSCRIPTS \
--	install-confexecDATA install-libLTLIBRARIES
-+install-exec-am: install-binSCRIPTS \
-+	install-libLTLIBRARIES
+-install-exec-am: install-binPROGRAMS install-libLTLIBRARIES \
++install-exec-am: install-binPROGRAMS \
+ 	install-nodist_binSCRIPTS
  
  install-html: install-html-recursive
- 
--- make/libs/libxml2/patches/110-pthread_fixes.libxml2.patch	2022-09-18 13:06:37.109155551 +0200
+++ make/libs/libxml2/patches/110-pthread_fixes.libxml2.patch	2022-09-21 17:02:11.312986847 +0200
@@ -1,6 +1,6 @@
 --- configure
 +++ configure
-@@ -15439,12 +15439,12 @@
+@@ -15994,12 +15994,12 @@
  	       if test "${THREAD_LIBS}" = "-lpthread" ; then
  	           if expr ${GCC_MEDIUM} \> 2 \& ${GCC_MAJOR} = 3 > /dev/null
  		   then
@@ -21,18 +21,18 @@
  Version: @VERSION@
  Description: libXML library version2.
  Requires:
--Libs: -L${libdir} -lxml2
--Libs.private: @ICU_LIBS@ @THREAD_LIBS@ @Z_LIBS@ @LZMA_LIBS@ @ICONV_LIBS@ @M_LIBS@ @WIN32_EXTRA_LIBADD@ @LIBS@
+-Libs: -L${libdir} @XML_LIBS@
+-Libs.private: @XML_PRIVATE_LIBS@ @LIBS@
 -Cflags: @XML_INCLUDEDIR@ @XML_CFLAGS@
 +Libs: -L${libdir} -lxml2 @ICU_LIBS@ @Z_LIBS@ @LZMA_LIBS@ @ICONV_LIBS@ @THREAD_LIBS@ @M_LIBS@ @WIN32_EXTRA_LIBADD@ @LIBS@
 +Cflags: @THREAD_CFLAGS@ @XML_INCLUDEDIR@ @XML_CFLAGS@
 --- libxml-2.0-uninstalled.pc.in
 +++ libxml-2.0-uninstalled.pc.in
-@@ -9,4 +9,4 @@
- Description: libXML library version2.
+@@ -10,4 +10,4 @@
  Requires:
- Libs: -L${libdir} -lxml2 @ICU_LIBS@ @THREAD_LIBS@ @Z_LIBS@ @LZMA_LIBS@ @ICONV_LIBS@ @M_LIBS@ @LIBS@
--Cflags: -I${includedir} @XML_INCLUDEDIR@ @XML_CFLAGS@
+ Libs: -L${libdir} @XML_LIBS@
+ Libs.private: @XML_PRIVATE_LIBS@ @LIBS@
+-Cflags: @XML_INCLUDEDIR@ @XML_CFLAGS@
 +Cflags: -I${includedir} @THREAD_CFLAGS@ @XML_INCLUDEDIR@ @XML_CFLAGS@
 --- xml2-config.in
 +++ xml2-config.in
--- make/dbus/dbus.mk	2022-09-18 13:06:37.073156000 +0200
+++ make/dbus/dbus.mk	2022-09-22 18:40:34.516891063 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, 1.13.18)
-$(PKG)_LIB_VERSION:=3.29.0
+$(call PKG_INIT_BIN, 1.15.0)
+$(PKG)_LIB_VERSION:=3
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=8078f5c25e34ab907ce06905d969dc8ef0ccbec367e1e1707c7ecf8460f4254e
+$(PKG)_HASH:=5073c8cb9ad20226647bb38f4965182b762a6e1f595ccdc8e59411014bfd640a
 $(PKG)_SITE:=http://$(pkg).freedesktop.org/releases/$(pkg)
 
 $(PKG)_STARTLEVEL=90 # before avahi
--- make/dbus/Config.in	2022-09-18 13:06:37.073156000 +0200
+++ make/dbus/Config.in	2022-09-22 18:47:29.891799209 +0200
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_DBUS
-	bool "D-Bus 1.13.18"
+	bool "D-Bus 1.15.0"
 	default n
 	select FREETZ_LIB_libexpat
 	select FREETZ_LIB_libpthread if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
--- make/dbus/external.files	2022-09-18 13:06:37.073156000 +0200
+++ make/dbus/external.files	2022-09-22 18:41:49.503484030 +0200
@@ -2,4 +2,4 @@
 [ "$EXTERNAL_FREETZ_PACKAGE_DBUS_monitor" == "y" ] && EXTERNAL_FILES+=" /usr/bin/dbus-monitor"
 [ "$EXTERNAL_FREETZ_PACKAGE_DBUS_send" == "y" ] && EXTERNAL_FILES+=" /usr/bin/dbus-send"
 [ "$EXTERNAL_FREETZ_PACKAGE_DBUS_uuidgen" == "y" ] && EXTERNAL_FILES+=" /usr/bin/dbus-uuidgen"
-[ "$EXTERNAL_FREETZ_LIB_libdbus" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libdbus-1.so.3.29.0"
+[ "$EXTERNAL_FREETZ_LIB_libdbus" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libdbus-1.so.3"
--- make/dbus/external.in.libs	2022-09-18 13:06:37.073156000 +0200
+++ make/dbus/external.in.libs	2022-09-22 18:42:22.146871970 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libdbus.so.3.29.0
+		 ${FREETZ_LIBRARY_DIR}/libdbus.so.3
 
--- make/dbus/patches/010-link_all_deplibs.patch	2022-09-18 13:06:37.073156000 +0200
+++ make/dbus/patches/010-link_all_deplibs.patch	2022-09-22 18:27:58.735225455 +0200
@@ -1,7 +1,7 @@
 revert Debian-2.4.2-1.2 changes
 --- configure
 +++ configure
-@@ -10941,7 +10941,7 @@
+@@ -11849,7 +11849,7 @@
      with_gnu_ld=no
      ;;
    linux* | k*bsd*-gnu | gnu*)
--- make/dbus/patches/020-drop_backtrace_dependency.patch	2022-09-18 13:06:37.073156000 +0200
+++ make/dbus/patches/020-drop_backtrace_dependency.patch	2022-09-22 18:50:54.831051803 +0200
@@ -1,18 +1,18 @@
---- configure
-+++ configure
-@@ -19812,6 +19812,7 @@
- 	fi
+--- configure	2022-09-22 18:31:21.515339000 +0200
++++ configure	2022-09-22 18:34:33.747688511 +0200
+@@ -20731,6 +20731,7 @@
+ 
  fi
  
 +if false; then
- for ac_header in execinfo.h
+        for ac_header in execinfo.h
  do :
-   ac_fn_c_check_header_mongrel "$LINENO" "execinfo.h" "ac_cv_header_execinfo_h" "$ac_includes_default"
-@@ -19880,6 +19881,7 @@
+   ac_fn_c_check_header_compile "$LINENO" "execinfo.h" "ac_cv_header_execinfo_h" "$ac_includes_default"
+@@ -20801,6 +20802,8 @@
  fi
  
  done
 +fi
- 
++
  
  # Add -D_POSIX_PTHREAD_SEMANTICS if on Solaris
