--- make/libs/freetype/freetype.mk	2022-08-11 17:31:33.555017000 +0200
+++ make/libs/freetype/freetype.mk	2022-08-12 07:05:32.487502455 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 2.10.1)
-$(PKG)_LIB_VERSION:=6.17.1
+$(call PKG_INIT_LIB, 2.12.1)
+$(PKG)_LIB_VERSION:=6
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=16dbfa488a21fe827dc27eaf708f42f7aa3bb997d745d31a19781628c36ba26f
+$(PKG)_HASH:=4766f20157cc4cf0cd292f80bf917f92d1c439b243ac3018debf6b9140c41a7f
 $(PKG)_SITE:=@SF/freetype
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/objs/.libs/libfreetype.so.$($(PKG)_LIB_VERSION)
--- make/libs/libpng/libpng.mk	2022-08-11 17:31:33.559017000 +0200
+++ make/libs/libpng/libpng.mk	2022-08-12 07:26:05.533833994 +0200
@@ -1,12 +1,12 @@
-$(call PKG_INIT_LIB, 1.2.59)
-$(PKG)_LIB_VERSION:=0.59.0
+$(call PKG_INIT_LIB, 1.6.38)
+$(PKG)_LIB_VERSION:=16.38.0
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=b4635f15b8adccc8ad0934eea485ef59cc4cae24d0f0300a9a941e51974ffcc7
+$(PKG)_HASH:=b3683e8b8111ebf6f1ac004ebb6b0c975cd310ec469d98364388e9cedbfa68be
 $(PKG)_SITE:=@SF/libpng
 
-$(PKG)_BINARY:=$($(PKG)_DIR)/.libs/libpng12.so.$($(PKG)_LIB_VERSION)
-$(PKG)_STAGING_BINARY:=$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/libpng12.so.$($(PKG)_LIB_VERSION)
-$(PKG)_TARGET_BINARY:=$($(PKG)_TARGET_DIR)/libpng12.so.$($(PKG)_LIB_VERSION)
+$(PKG)_BINARY:=$($(PKG)_DIR)/.libs/libpng16.so.$($(PKG)_LIB_VERSION)
+$(PKG)_STAGING_BINARY:=$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/libpng16.so.$($(PKG)_LIB_VERSION)
+$(PKG)_TARGET_BINARY:=$($(PKG)_TARGET_DIR)/libpng16.so.$($(PKG)_LIB_VERSION)
 
 $(PKG)_DEPENDS_ON += zlib
 
@@ -31,9 +31,9 @@
 		DESTDIR="$(TARGET_TOOLCHAIN_STAGING_DIR)" \
 		install
 	$(PKG_FIX_LIBTOOL_LA) \
-		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/libpng12.la \
-		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/pkgconfig/libpng12.pc \
-		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/bin/libpng12-config
+		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/libpng16.la \
+		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/pkgconfig/libpng16.pc \
+		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/bin/libpng16-config
 
 $($(PKG)_TARGET_BINARY): $($(PKG)_STAGING_BINARY)
 	$(INSTALL_LIBRARY_STRIP)
@@ -48,7 +48,7 @@
 		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/pkgconfig/libpng*.pc \
 		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/bin/libpng*-config \
 		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/include/png*.h \
-		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/include/libpng12 \
+		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/include/libpng16 \
 		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/share/man/man3/libpng*.3 \
 		$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/share/man/man5/png.5
 
--- make/libs/libpng/Config.in	2022-08-11 17:31:33.559017000 +0200
+++ make/libs/libpng/Config.in	2022-08-12 07:26:42.833464754 +0200
@@ -1,4 +1,4 @@
-config FREETZ_LIB_libpng12
+config FREETZ_LIB_libpng16
 	bool "PNG library (libpng.so)"
 	select FREETZ_LIB_libm if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libz
--- make/libs/libpng/external.files	2022-08-11 17:31:33.559017000 +0200
+++ make/libs/libpng/external.files	2022-08-12 07:26:21.585674663 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libpng12" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libpng12.so.0.59.0"
+[ "$EXTERNAL_FREETZ_LIB_libpng16" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libpng16.so.16.37.0"
--- make/libs/libpng/external.in	2022-08-11 17:31:33.559017000 +0200
+++ make/libs/libpng/external.in	2022-08-12 07:27:37.500929503 +0200
@@ -1,8 +1,8 @@
-config EXTERNAL_FREETZ_LIB_libpng12
-	depends on EXTERNAL_ENABLED && FREETZ_LIB_libpng12
-	bool "libpng12"
+config EXTERNAL_FREETZ_LIB_libpng16
+	depends on EXTERNAL_ENABLED && FREETZ_LIB_libpng16
+	bool "libpng16"
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libpng12.so.0.59.0
+		 ${FREETZ_LIBRARY_DIR}/libpng16.so.16.37.0
 
--- make/libs/libpng/Makefile.in	2022-08-11 17:31:33.559017000 +0200
+++ make/libs/libpng/Makefile.in	2022-08-12 07:28:45.428273299 +0200
@@ -1,3 +1,3 @@
-ifeq ($(strip $(FREETZ_LIB_libpng12)),y)
+ifeq ($(strip $(FREETZ_LIB_libpng16)),y)
 LIBS+=libpng
 endif
--- make/rrdtool/Config.in.libs	2022-08-11 17:31:33.651015000 +0200
+++ make/rrdtool/Config.in.libs	2022-08-12 07:29:57.087590412 +0200
@@ -3,7 +3,7 @@
 	select FREETZ_LIB_libart_lgpl_2
 	select FREETZ_LIB_libfreetype
 	select FREETZ_LIB_libm        if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
-	select FREETZ_LIB_libpng12
+	select FREETZ_LIB_libpng16
 	select FREETZ_LIB_libz
 	default n
 	help
@@ -15,7 +15,7 @@
 	select FREETZ_LIB_libart_lgpl_2
 	select FREETZ_LIB_libfreetype
 	select FREETZ_LIB_libm        if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
-	select FREETZ_LIB_libpng12
+	select FREETZ_LIB_libpng16
 	select FREETZ_LIB_libz
 	default n
 	help
--- make/netpbm/Config.in	2022-08-11 17:31:33.631015000 +0200
+++ make/netpbm/Config.in	2022-08-12 07:53:14.675042981 +0200
@@ -30,7 +30,7 @@
 config FREETZ_PACKAGE_NETPBM_PNGTOPNM
 	bool "Include pngtopnm"
 	depends on FREETZ_PACKAGE_NETPBM
-	select FREETZ_LIB_libpng12
+	select FREETZ_LIB_libpng16
 	select FREETZ_LIB_libz
 	default y
 	help
@@ -47,7 +47,7 @@
 config FREETZ_PACKAGE_NETPBM_PNMTOPNG
 	bool "Include pnmtopng"
 	depends on FREETZ_PACKAGE_NETPBM
-	select FREETZ_LIB_libpng12
+	select FREETZ_LIB_libpng16
 	select FREETZ_LIB_libz
 	default y
 	help
--- make/libs/libgd/Config.in	2022-09-18 09:29:13.551135000 +0200
+++ make/libs/libgd/Config.in	2022-09-18 09:35:49.544850868 +0200
@@ -3,7 +3,7 @@
 	default n
 	select FREETZ_LIB_libjpeg
 	select FREETZ_LIB_libfreetype
-	select FREETZ_LIB_libpng12
+	select FREETZ_LIB_libpng16
 	select FREETZ_LIB_libz
 	help
 		GD is an open source code library for the dynamic creation of images by
--- make/libs/libffi/libffi.mk	2022-09-18 13:06:37.101155000 +0200
+++ make/libs/libffi/libffi.mk	2022-09-21 19:18:55.078241188 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 3.2.1)
-$(PKG)_LIB_VERSION:=6.0.4
+$(call PKG_INIT_LIB, 3.4.3)
+$(PKG)_LIB_VERSION:=8
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=d06ebb8e1d9a22d19e38d63fdb83954253f39bedc5d46232a05645685722ca37
+$(PKG)_HASH:=4416dd92b6ae8fcb5b10421e711c4d3cb31203d77521a77d85d0102311e6c3b8
 $(PKG)_SITE:=ftp://sourceware.org/pub/libffi
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/$(TARGET_GNU_TRIPLET)/.libs/libffi.so.$($(PKG)_LIB_VERSION)
--- make/libs/libffi/external.in	2022-09-18 13:06:37.101155000 +0200
+++ make/libs/libffi/external.in	2022-09-21 19:27:14.836040918 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libffi.so.6.0.4
+		 ${FREETZ_LIBRARY_DIR}/libffi.so.8
 
--- make/libs/libffi/external.files	2022-09-18 13:06:37.101155000 +0200
+++ make/libs/libffi/external.files	2022-09-21 19:26:58.136371614 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libffi" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libffi.so.6.0.4"
+[ "$EXTERNAL_FREETZ_LIB_libffi" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libffi.so.8"
--- make/libs/libffi/patches/010-no-tests-no-docs.libffi.patch	2022-09-18 13:06:37.101155000 +0200
+++ make/libs/libffi/patches/010-no-tests-no-docs.libffi.patch	2022-09-21 19:04:03.451910963 +0200
@@ -1,20 +1,11 @@
 --- Makefile.in
 +++ Makefile.in
-@@ -606,7 +606,7 @@
+@@ -526,7 +526,7 @@
  top_srcdir = @top_srcdir@
  AUTOMAKE_OPTIONS = foreign subdir-objects
  ACLOCAL_AMFLAGS = -I m4
--SUBDIRS = include testsuite man
-+SUBDIRS = include
- EXTRA_DIST = LICENSE ChangeLog.v1 ChangeLog.libgcj			\
- 	 src/aarch64/ffi.c src/aarch64/ffitarget.h src/aarch64/sysv.S	\
- 	 src/alpha/ffi.c src/alpha/osf.S			\
-@@ -1998,7 +1998,7 @@
- 
- info-am: $(INFO_DEPS)
- 
--install-data-am: install-info-am install-pkgconfigDATA
-+install-data-am: install-pkgconfigDATA
- 
- install-dvi: install-dvi-recursive
- 
+-SUBDIRS = include testsuite man $(am__append_1)
++SUBDIRS = include $(am__append_1)
+ EXTRA_DIST = LICENSE ChangeLog.old					\
+ 	m4/libtool.m4 m4/lt~obsolete.m4					\
+ 	 m4/ltoptions.m4 m4/ltsugar.m4 m4/ltversion.m4			\
--- make/libs/libffi/patches/020-includedir.libffi.patch	2022-09-18 13:06:37.101155000 +0200
+++ make/libs/libffi/patches/020-includedir.libffi.patch	2022-09-21 19:17:23.844202409 +0200
@@ -1,24 +1,6 @@
---- include/Makefile.in
-+++ include/Makefile.in
-@@ -314,7 +314,7 @@
- AUTOMAKE_OPTIONS = foreign
- DISTCLEANFILES = ffitarget.h
- EXTRA_DIST = ffi.h.in ffi_common.h
--includesdir = $(libdir)/@PACKAGE_NAME@-@PACKAGE_VERSION@/include
-+includesdir = @includedir@
- nodist_includes_HEADERS = ffi.h ffitarget.h
- all: all-am
- 
 --- libffi.pc.in
 +++ libffi.pc.in
-@@ -1,11 +1,10 @@
- prefix=@prefix@
- exec_prefix=@exec_prefix@
- libdir=@libdir@
--toolexeclibdir=@toolexeclibdir@
--includedir=${libdir}/@PACKAGE_NAME@-@PACKAGE_VERSION@/include
-+includedir=@includedir@
- 
+@@ -7,5 +7,5 @@
  Name: @PACKAGE_NAME@
  Description: Library supporting Foreign Function Interfaces
  Version: @PACKAGE_VERSION@
--- make/libs/bluez-libs/bluez-libs.mk	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/bluez-libs/bluez-libs.mk	2022-09-23 16:22:24.928804000 +0200
@@ -1,8 +1,8 @@
-$(call PKG_INIT_LIB, 2.25)
-$(PKG)_LIB_VERSION:=1.0.25
+$(call PKG_INIT_LIB, 3.36)
+$(PKG)_LIB_VERSION:=2
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=0acce2bac854588d2597cbcdb60a6ec4de5fe3e16d224bdfab8f2dd96ad242fb
-$(PKG)_SITE:=@SF/bluez
+$(PKG)_HASH:=692f994d8479143c18e781f9afa6c8265770a5affbf2e6147edc70b78aa3bfae
+$(PKG)_SITE:=http://bluez.sf.net/download/
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/src/.libs/libbluetooth.so.$($(PKG)_LIB_VERSION)
 $(PKG)_STAGING_BINARY:=$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/libbluetooth.so.$($(PKG)_LIB_VERSION)
@@ -43,4 +43,4 @@
 $(pkg)-uninstall:
 	$(RM) $(BLUEZ_LIBS_TARGET_DIR)/libbluetooth*.so*
 
-$(PKG_FINISH)
\ Kein Zeilenumbruch am Dateiende.
+$(PKG_FINISH)
--- make/libs/bluez-libs/external.in	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/bluez-libs/external.in	2022-09-23 16:28:04.701545495 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libbluetooth.so.1.0.25
+		 ${FREETZ_LIBRARY_DIR}/libbluetooth.so.2
 
--- make/libs/bluez-libs/external.files 2022-09-22 19:57:32.295197000 +0200
+++ make/libs/bluez-libs/external.files	2022-09-23 16:27:59.009632745 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libbluetooth" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libbluetooth.so.1.0.25"
+[ "$EXTERNAL_FREETZ_LIB_libbluetooth" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libbluetooth.so.2"
--- make/libs/gettext/gettext.mk	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/gettext/gettext.mk	2022-09-23 18:25:51.787222797 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 0.18.1.1)
-$(PKG)_LIB_VERSION:=8.1.1
-$(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=93ac71a7afa5b70c1b1032516663658c51e653087f42a3fc8044752c026443e9
+$(call PKG_INIT_LIB, 0.21)
+$(PKG)_LIB_VERSION:=8
+$(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
+$(PKG)_HASH:=d20fcbb537e02dcf1383197ba05bd0734ef7bf5db06bdb241eb69b7d16b73192
 $(PKG)_SITE:=@GNU/$(pkg)
 
 # we only want libintl
--- make/libs/gettext/external.in	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/gettext/external.in	2022-09-23 18:31:28.995560008 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libintl.so.8.1.1
+		 ${FREETZ_LIBRARY_DIR}/libintl.so.8
 
--- make/libs/gettext/external.files	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/gettext/external.files	2022-09-23 18:31:17.919679827 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libintl" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libintl.so.8.1.1"
+[ "$EXTERNAL_FREETZ_LIB_libintl" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libintl.so.8"
--- make/libs/gsasl/gsasl.mk	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/gsasl/gsasl.mk	2022-09-24 09:11:48.720060864 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 1.10.0)
-$(PKG)_LIB_VERSION:=7.10.0
+$(call PKG_INIT_LIB, 2.2.0)
+$(PKG)_LIB_VERSION:=18
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=85bcbd8ee6095ade7870263a28ebcb8832f541ea7393975494926015c07568d3
+$(PKG)_HASH:=79b868e3b9976dc484d59b29ca0ae8897be96ce4d36d32aed5d935a7a3307759
 $(PKG)_SITE:=@GNU/$(pkg)
 
 $(PKG)_LIBNAME_SHORT := lib$(pkg)
--- make/libs/jpeg/jpeg.mk	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/jpeg/jpeg.mk	2022-09-24 09:26:10.757851640 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 9d)
-$(PKG)_LIB_VERSION:=9.4.0
+$(call PKG_INIT_LIB, 9e)
+$(PKG)_LIB_VERSION:=9
 $(PKG)_SOURCE:=jpegsrc.v$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=6c434a3be59f8f62425b2e3c077e785c9ce30ee5874ea1c270e843f273ba71ee
+$(PKG)_HASH:=4077d6a6a75aeb01884f708919d25934c93305e49f7e3f36db9129320e6f4f3d
 $(PKG)_SITE:=http://ijg.org/files
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/.libs/libjpeg.so.$($(PKG)_LIB_VERSION)
--- make/libs/jpeg/external.in	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/jpeg/external.in	2022-09-24 09:27:45.155536259 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libjpeg.so.9.4.0
+		 ${FREETZ_LIBRARY_DIR}/libjpeg.so.9
 
--- make/libs/jpeg/external.files	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/jpeg/external.files	2022-09-24 09:27:55.467292907 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libjpeg" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libjpeg.so.9.4.0"
+[ "$EXTERNAL_FREETZ_LIB_libjpeg" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libjpeg.so.9"
--- make/libs/ldns/ldns.mk	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/ldns/ldns.mk	2022-09-24 09:58:08.599736270 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 1.7.1)
-$(PKG)_LIB_VERSION:=3.0.0
+$(call PKG_INIT_LIB, 1.8.3)
+$(PKG)_LIB_VERSION:=3
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=8ac84c16bdca60e710eea75782356f3ac3b55680d40e1530d7cea474ac208229
+$(PKG)_HASH:=c3f72dd1036b2907e3a56e6acf9dfb2e551256b3c1bbd9787942deeeb70e7860
 $(PKG)_SITE:=https://www.nlnetlabs.nl/downloads/$(pkg)/
 $(PKG)_BINARY:=$($(PKG)_DIR)/.libs/lib$(pkg).so.$($(PKG)_LIB_VERSION)
 $(PKG)_STAGING_BINARY:=$(TARGET_TOOLCHAIN_STAGING_DIR)/usr/lib/lib$(pkg).so.$($(PKG)_LIB_VERSION)
--- make/libs/ldns/external.in	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/ldns/external.in	2022-09-24 09:59:10.734783847 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libldns.so.3.0.0
+		 ${FREETZ_LIBRARY_DIR}/libldns.so.3
 
--- make/libs/ldns/external.files	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/ldns/external.files	2022-09-24 09:59:03.882888795 +0200
@@ -1,2 +1,2 @@
-[ "$EXTERNAL_FREETZ_LIB_libldns" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libldns.so.3.0.0"
+[ "$EXTERNAL_FREETZ_LIB_libldns" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libldns.so.3"

--- make/libs/db/db.mk	2022-09-24 19:44:27.155340000 +0200
+++ make/libs/db/db.mk	2022-09-28 18:17:34.283845106 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 4.8.30)
-$(PKG)_LIB_VERSION:=4.8
+$(call PKG_INIT_LIB, 18.1.40)
+$(PKG)_LIB_VERSION:=18.1
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=e0491a07cdb21fb9aa82773bbbedaeb7639cbd0e7f96147ab46141e0045db72a
+$(PKG)_HASH:=0cecb2ef0c67b166de93732769abdeba0555086d51de1090df325e18ee8da9c8
 $(PKG)_SITE:=http://download.oracle.com/berkeley-db
 
 $(PKG)_LIBNAME:=lib$(pkg)-$($(PKG)_LIB_VERSION).so
--- make/libs/db/external.files	2022-09-24 19:44:27.155340000 +0200
+++ make/libs/db/external.files	2022-09-28 18:17:28.019957180 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libdb" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libdb-4.8.so"
+[ "$EXTERNAL_FREETZ_LIB_libdb" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libdb-18.1.so"
--- make/libs/db/external.in	2022-09-24 19:44:27.155340000 +0200
+++ make/libs/db/external.in	2022-09-28 18:17:50.043563252 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libdb-4.8.so
+		 ${FREETZ_LIBRARY_DIR}/libdb-18.1.so
 
--- make/libs/db/patches/100-no_docs.db.patch	2022-09-24 19:44:27.155340000 +0200
+++ make/libs/db/patches/100-no_docs.db.patch	2022-09-28 18:14:16.331403337 +0200
@@ -1,13 +1,13 @@
---- dist/Makefile.in	2009-12-18 23:26:01.000000000 +0100
-+++ dist/Makefile.in	2010-01-16 22:46:48.000000000 +0100
-@@ -925,9 +925,9 @@
+--- dist/Makefile.in
++++ dist/Makefile.in
+@@ -1215,9 +1215,9 @@
  # Library and standard utilities install.
  ##################################################
  library_install: install_setup
--library_install: install_include install_lib install_utilities install_docs
+-library_install: install_include install_lib install_utilities @ADDITIONAL_INSTALL@ install_docs
 +library_install: install_include install_lib
  
--uninstall: uninstall_include uninstall_lib uninstall_utilities uninstall_docs
+-uninstall: uninstall_include uninstall_lib uninstall_utilities @ADDITIONAL_UNINSTALL@ uninstall_docs
 +uninstall: uninstall_include uninstall_lib
  
  install_setup:
--- make/libs/jansson/jansson.mk	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/jansson/jansson.mk	2022-09-29 16:36:39.757383588 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 2.7)
-$(PKG)_LIB_VERSION:=4.7.0
+$(call PKG_INIT_LIB, 2.13)
+$(PKG)_LIB_VERSION:=4
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=7905e6590fb316c0ff943df3dc6a21cd81a59cff7a6d12514054c359d04d78d7
+$(PKG)_HASH:=02c31bc16e702b30feb06d18bbfe086c0d8c938e906950980af7adcdb324541b
 $(PKG)_SITE:=http://www.digip.org/jansson/releases
 
 $(PKG)_LIBNAME:=lib$(pkg).so.$($(PKG)_LIB_VERSION)
--- make/libs/jansson/external.in	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/jansson/external.in	2022-09-29 16:37:41.968117808 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libjansson.so.4.7.0
+		 ${FREETZ_LIBRARY_DIR}/libjansson.so.4
 
--- make/libs/jansson/external.files	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/jansson/external.files	2022-09-29 16:37:34.024274252 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libjansson" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libjansson.so.4.7.0"
+[ "$EXTERNAL_FREETZ_LIB_libjansson" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libjansson.so.4"
--- make/libs/libcap/libcap.mk	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/libcap/libcap.mk	2022-09-29 17:07:16.519005435 +0200
@@ -1,10 +1,10 @@
-$(call PKG_INIT_LIB, $(if $(FREETZ_AVM_GCC_4_MAX),2.49,2.65))
+$(call PKG_INIT_LIB, $(if $(FREETZ_AVM_GCC_4_MAX),2.49,2.66))
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
 $(PKG)_HASH_ABANDON:=e98bc4d93645082ec787730b0fd1a712b38882465c505777de17c338831ee181
-$(PKG)_HASH_CURRENT:=73e350020cc31fe15360879d19384ffa3395a825f065fcf6bda3a5cdf965bebd
+$(PKG)_HASH_CURRENT:=15c40ededb3003d70a283fe587a36b7d19c8b3b554e33f86129c059a4bb466b2
 $(PKG)_HASH:=$($(PKG)_HASH_$(if $(FREETZ_AVM_GCC_4_MAX),ABANDON,CURRENT))
 $(PKG)_SITE:=@KERNEL/linux/libs/security/linux-privs/libcap2
-### VERSION:=2.49/2.65
+### VERSION:=2.49/2.66
 ### WEBSITE:=https://sites.google.com/site/fullycapable/
 ### MANPAGE:=https://pkg.go.dev/kernel.org/pub/linux/libs/security/libcap/cap
 ### CHANGES:=https://sites.google.com/site/fullycapable/release-notes-for-libcap
--- make/libs/libcap/external.files	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/libcap/external.files	2022-09-29 17:02:57.031597214 +0200
@@ -1,3 +1,3 @@
-[ "FREETZ_AVM_GCC_4_MAX" == "y" ] && ELIBVER="2.49" || ELIBVER="2.65"
+[ "FREETZ_AVM_GCC_4_MAX" == "y" ] && ELIBVER="2.49" || ELIBVER="2.66"
 
 [ "$EXTERNAL_FREETZ_LIB_libcap" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libcap.so.$ELIBVER"
--- make/libs/libev/libev.mk	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/libev/libev.mk	2022-09-29 17:45:10.288375120 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 4.15)
+$(call PKG_INIT_LIB, 4.33)
 $(PKG)_SHLIB_VERSION:=4.0.0
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=b2dd43a073556f5350cbed05b6ef444dcc4b563f4e0b1009d7bf448261606feb
+$(PKG)_HASH:=507eb7b8d1015fbec5b935f34ebed15bf346bed04a11ab82b8eee848c4205aea
 $(PKG)_SITE:=http://dist.schmorp.de/libev
 
 $(PKG)_LIBNAME=$(pkg).so.$($(PKG)_SHLIB_VERSION)
--- make/libs/libev/patches/010-no_man.libev.patch	2022-09-22 19:44:02.398094000 +0200
+++ make/libs/libev/patches/010-no_man.libev.patch	2022-09-29 17:31:25.130036899 +0200
@@ -1,6 +1,6 @@
 --- Makefile.in
 +++ Makefile.in
-@@ -751,7 +751,7 @@
+@@ -849,7 +849,7 @@
  
  info-am:
  
--- make/libs/libev/patches/020-no_libevent_compat_header.libev.patch	2022-09-22 19:44:02.398094000 +0200
+++ make/libs/libev/patches/020-no_libevent_compat_header.libev.patch	2022-09-29 17:31:25.146036495 +0200
@@ -1,6 +1,6 @@
 --- Makefile.in
 +++ Makefile.in
-@@ -267,7 +267,7 @@
+@@ -357,7 +357,7 @@
  	     ev.3 ev.pod Symbols.ev Symbols.event
  
  man_MANS = ev.3
--- make/libs/libev/patches/030-verbose_backend.libev.patch	2022-09-22 19:44:02.398094000 +0200
+++ make/libs/libev/patches/030-verbose_backend.libev.patch	2022-09-29 17:38:07.880820458 +0200
@@ -1,6 +1,6 @@
 --- ev.c
 +++ ev.c
-@@ -2381,21 +2381,27 @@
+@@ -3078,15 +3078,19 @@
  /*****************************************************************************/
  
  #if EV_USE_IOCP
@@ -19,6 +19,10 @@
 +# warning using "ev_epoll.c"
  # include "ev_epoll.c"
  #endif
+ #if EV_USE_LINUXAIO
+@@ -3096,9 +3100,11 @@
+ # include "ev_iouring.c"
+ #endif
  #if EV_USE_POLL
 +# warning using "ev_poll.c"
  # include "ev_poll.c"
@@ -27,4 +31,3 @@
 +# warning using "ev_select.c"
  # include "ev_select.c"
  #endif
- 
