--- tools/make/yf-bootmanager-host/yf-bootmanager-host.mk.org	2022-06-26 06:51:58.469159745 +0200
+++ tools/make/yf-bootmanager-host/yf-bootmanager-host.mk	2022-06-26 07:06:50.455682928 +0200
@@ -1,8 +1,8 @@
 YF_BOOTMANAGER_REPOSITORY:=https://github.com/PeterPawn/YourFritz.git
-$(call TOOL_INIT, $(shell git ls-remote --tags $(YF_BOOTMANAGER_REPOSITORY) freetz-ng-version | sed -n -e "s|^\([0-9a-f]*\).*|\1|p"))
+$(call TOOLS_INIT, $(shell git ls-remote --tags $(YF_BOOTMANAGER_REPOSITORY) freetz-ng-test | sed -n -e "s|^\([0-9a-f]*\).*|\1|p"))
 # Versions after this commit have no vanilla GPL3 - but it's not a problem to use the unchanged(!) version in an own image.
-$(TOOL)_SITE:=git_sparse@https://github.com/PeterPawn/YourFritz.git,bootmanager
-$(TOOL)_SOURCE:=yf-bootmanager-$($(TOOL)_VERSION).tar.xz
+$(PKG)_SITE:=git_sparse@https://github.com/PeterPawn/YourFritz.git,bootmanager
+$(PKG)_SOURCE:=yf-bootmanager-$($(PKG)_VERSION).tar.xz
 
 
 $(TOOLS_SOURCE_DOWNLOAD)
