--- config/mod/source.in	2023-09-16 09:07:19.487294534 +0200
+++ config/mod/source.in	2023-09-16 09:07:42.998981511 +0200
@@ -39,7 +39,6 @@
 	depends on ! FREETZ_KERNEL_AVM_CHAOTIC_PACK
 	depends on ! FREETZ_KERNEL_VERSION_4_1_38                                              # 07_1X-final 7581/7582 (corrupt)
 	depends on !(FREETZ_AVM_SOURCE_FOR_SYSTEM_TYPE_MAPLE && FREETZ_AVM_VERSION_07_2X_MAX)  # 07_2X-final 7510/1209 (corrupt)
-	depends on ! FREETZ_KERNEL_VERSION_4_19_183                                            # 07_5X-final 7539 (corrupt)
 	default y
 	help
 		There are no avm sources at all of this kernel version and thus no avmdiff, so unpatched vanilla is used.
@@ -48,8 +47,7 @@
 	bool
 	depends on \
 		FREETZ_AVM_SOURCE_7582_07_15 || \
-		FREETZ_AVM_SOURCE_1209_07_30 || \
-		FREETZ_AVM_SOURCE_7539_07_56
+		FREETZ_AVM_SOURCE_1209_07_30
 	default y
 	help
 		AVM used the "chaos theory" packing method and so these sources would waste to much time to integrate.
