--- patches/scripts/000-remove_i-matik_orig_files.sh	1970-01-01 01:00:00.000000000 +0100
+++ patches/scripts/000-remove_i-matik_orig_files.sh	2023-12-05 17:22:00.210913615 +0100
@@ -0,0 +1,7 @@
+echo1 "removing i-matik .orig files"
+
+rm_files "${FILESYSTEM_MOD_DIR}/etc/default.mod/motd.orig"
+rm_files "${FILESYSTEM_MOD_DIR}/usr/lib/cgi-bin/mod/info.cgi.orig"
+rm_files "${FILESYSTEM_MOD_DIR}/usr/mww/cgi-bin/status.d/10-box.sh.orig"
+rm_files "${FILESYSTEM_MOD_DIR}/usr/lib/mww/cgi/menu-new.sh.orig"
+
