--- patches/cond/531-enable_gpon_serial/531-enable_gpon_serial_07_5X.patch	1970-01-01 01:00:00.000000000 +0100
+++ patches/cond/531-enable_gpon_serial/531-enable_gpon_serial_07_5X.patch	2023-12-25 18:53:36.850479367 +0100
@@ -0,0 +1,11 @@
+--- usr/www/all/support.lua
++++ usr/www/all/support.lua
+@@ -718,7 +718,7 @@
+ --end
+ end
+ function gpon_serial.show()
+-return config.GPON and (config.country ~= "049" or g_is_internal_or_inhaus_version)
++return true
+ end
+ function gpon_serial.dosave()
+ local saveset = {}
--- patches/scripts/531-enable_gpon_serial.sh	2023-11-04 13:42:27.848150174 +0100
+++ patches/scripts/531-enable_gpon_serial.sh	2023-12-25 18:55:05.812602261 +0100
@@ -1,8 +1,9 @@
 [ "$FREETZ_ENABLE_GPON_SERIAL" == "y" ] || return 0
 echo1 "re-adding GPON serial textbox"
 
-replace_lua_function \
-  $HTML_LANG_MOD_DIR/support.lua \
-  "gpon_serial.show" \
-  "config.GPON"
+if [ "$FREETZ_AVM_VERSION_07_5X_MIN" == "y" ] ; then
+	PVER="07_5X"
+fi
+
+modpatch "$FILESYSTEM_MOD_DIR" "${PATCHES_COND_DIR}/531-enable_gpon_serial/531-enable_gpon_serial_${PVER}.patch"
 
