--- config/ui/patches.in	2024-03-23 08:51:40.903774246 +0100
+++ config/ui/patches.in	2024-03-25 12:04:43.141426334 +0100
@@ -1270,6 +1270,38 @@
 		 812-show_update_config.sh
 		 821-show_network_repeater.sh
 		 822-show_dvb_services.sh
+ 
+config FREETZ_PATCH_EXTENDER_INTERNET
+	bool "Enable internet and certificate settings in AVM user web interface"
+	depends on FREETZ_TYPE_EXTENDER && \
+	    ( \
+		FREETZ_AVM_VERSION_07_2X_MIN || \
+		FREETZ_SHOW_DEVELOPER \
+		) && ( \
+		FREETZ_TYPE_1200 || \
+		FREETZ_TYPE_1209 || \
+		FREETZ_TYPE_1750 || \
+		FREETZ_TYPE_1759 || \
+		FREETZ_TYPE_2400 || \
+		FREETZ_TYPE_6000 || \
+		FREETZ_SHOW_DEVELOPER \
+		)
+	default n
+	help
+		Enables hidden menu for internet and certificate settings
+		in AVM user web interface for wifi extenders.
+		This allows the upload of custom user certificates instead
+		of box's self signed  certificates for secure https traffic.
+		Tested for repeaters 1200AX, 1750E, 2400 and 6000 with Firmware > 7.2X.
+		Developer can activate this patch at own risk also
+		for other (untested) repeaters and firmwares.
+
+config FREETZ_PATCH_FEEDBACK
+	bool "Patch web menu feedback message"
+	depends on FREETZ_AVM_VERSION_07_5X_MIN && (FREETZ_TYPE_FIRMWARE_LABOR || FREETZ_TYPE_FIRMWARE_PLUS || FREETZ_TYPE_FIRMWARE_INHAUS)
+	default n
+	help
+		Hides the "feedback send" message from the web interface.
 
 config FREETZ_PATCH_SIGNED
 	bool "Patch web menu signed message"
@@ -1313,6 +1345,16 @@
 		help
 			noUpdateSearch does not trigger the message.
 
+	if FREETZ_PATCH_SECURE_nus
+
+		config FREETZ_PATCH_SECURE_au
+			bool "Caused by Autoupdate"
+			depends on FREETZ_AVM_VERSION_07_8X_MIN
+			default y
+			help
+				Hides Autoupdate does not trigger the message.
+	endif # FREETZ_PATCH_SECURE_nus
+
 endif # FREETZ_PATCH_SECURE
 
 config FREETZ_MODIFY_DSL_WARNING
@@ -1347,8 +1389,13 @@
 
 config FREETZ_ENABLE_GPON_SERIAL
 	bool "Enable GPON serial textbox"
-	depends on FREETZ_TYPE_FIBER && FREETZ_AVM_VERSION_07_5X_MAX
+	depends on FREETZ_AVM_VERSION_07_5X_MIN && !FREETZ_TYPE_FIRMWARE_INHAUS
 	depends on !FREETZ_REMOVE_SUPPORT
+	depends on \
+		( FREETZ_AVM_VERSION_07_5X_MIN && ( \
+			FREETZ_TYPE_5530 || \
+			FREETZ_TYPE_5590 \
+		))
 	default n
 	help
 		Re-enables access to the GPON serial textbox on support page.
@@ -1374,7 +1421,7 @@
 menuconfig FREETZ_PATCH_MODFS_BOOT_MANAGER
 	bool "Add yf-bootmanager"
 	depends on FREETZ_AVM_VERSION_07_8X_MAX
-	depends on !FREETZ_ADD_INACTIVE_SYSTEM || (FREETZ_ADD_INACTIVE_SYSTEM && FREETZ_AVM_PROP_INNER_FILESYSTEM_TYPE_CPIO)
+	depends on !FREETZ_ADD_INACTIVE_SYSTEM || (FREETZ_ADD_INACTIVE_SYSTEM && FREETZ_AVM_PROP_INNER_FILESYSTEM_TYPE_CPIO || FREETZ_PACKAGE_DTC_fitdump)
 	depends on FREETZ_AVM_VERSION_06_5X_MIN
 	depends on ( FREETZ_AVM_HAS_SEPARATE_FILESYSTEM_IMAGE && !FREETZ_AVM_HAS_FWLAYOUT_5 ) || \
 		                   ( FREETZ_AVM_HAS_FWLAYOUT_5 && (FREETZ_AVM_HAS_BLK_DEV_LOOP_BUILTIN || FREETZ_REPLACE_MODULE_AVAILABLE) )
--- patches/scripts/193-patch_webmenu_feedback.sh	1970-01-01 01:00:00.000000000 +0100
+++ patches/scripts/193-patch_webmenu_feedback.sh	2024-03-25 12:04:43.141426334 +0100
@@ -0,0 +1,12 @@
+[ "$FREETZ_PATCH_FEEDBACK" == "y" ] || return 0
+echo1 "applying webmenu feedback patch"
+
+for oem in $(supported_brandings) all; do
+	[ -d "${FILESYSTEM_MOD_DIR}/usr/www/${oem}" -a ! -L "${FILESYSTEM_MOD_DIR}/usr/www/${oem}" ] || continue
+
+	for file in /usr/www/${oem}/home/home.js; do
+		[ ! -e ${FILESYSTEM_MOD_DIR}${file} ] && continue
+		[ "$FREETZ_PATCH_FEEDBACK" == "y" ] && modsed 's/data[.]fritzos[.]isLabor/false/g'   ${FILESYSTEM_MOD_DIR}${file}
+	done
+done
+
--- patches/scripts/194-patch_webmenu_secure.sh	2023-11-08 17:31:48.206710457 +0100
+++ patches/scripts/194-patch_webmenu_secure.sh	2024-03-25 11:56:03.767326000 +0100
@@ -10,6 +10,7 @@
 		[ "$FREETZ_PATCH_SECURE_sip" == "y" ] && modsed 's/data[.]fritzos[.]ipphone_from_outside/false/g' ${FILESYSTEM_MOD_DIR}${file}
 		[ "$FREETZ_PATCH_SECURE_nau" == "y" ] && modsed 's/data[.]fritzos[.]noAutoupdate/false/g'         ${FILESYSTEM_MOD_DIR}${file}
 		[ "$FREETZ_PATCH_SECURE_nus" == "y" ] && modsed 's/data[.]fritzos[.]noUpdateSearch/false/g'       ${FILESYSTEM_MOD_DIR}${file}
+		[ "$FREETZ_PATCH_SECURE_au" == "y" ] && modsed 's/data[.]fritzos[.]Autoupdate/false/g'       ${FILESYSTEM_MOD_DIR}${file}
 	done
 done
 
--- patches/scripts/823-show_internet_extender.sh	1970-01-01 01:00:00.000000000 +0100
+++ patches/scripts/823-show_internet_extender.sh	2024-03-25 12:04:43.141426334 +0100
@@ -0,0 +1,9 @@
+[ "$FREETZ_PATCH_EXTENDER_INTERNET" == "y" ] || return 0
+[ "$FREETZ_TYPE_EXTENDER" == "y" ] || return 0
+echo1 "enabling internet and certificate config for wifi extender"
+
+# patcht Internet > Freigaben
+
+modsed \
+  's/\(.*\)\(\["meshSet"\] = {.*\)/\1\["remoteHttps"\] = config\.REMOTE_HTTPS and {\n\["show"\] = true,\n\["lua"\] = "internet\/remote_https\.lua",\n\["help"\] = \(forLuaOnly and "hilfe_internet_remote_https"\) or true\n} or nil,\n\2/' \
+  "${MENU_DATA_LUA}"
