--- config/ui/patches.in	2023-08-22 21:37:18.845440609 +0200
+++ config/ui/patches.in	2023-11-08 17:28:16.263690699 +0100
@@ -1270,6 +1270,31 @@
 		 812-show_update_config.sh
 		 821-show_network_repeater.sh
 		 822-show_dvb_services.sh
+ 
+config FREETZ_PATCH_EXTENDER_INTERNET
+	bool "Enable internet and certificate settings in AVM user web interface"
+	depends on FREETZ_TYPE_EXTENDER && \
+	    ( \
+		FREETZ_AVM_VERSION_07_2X_MIN || \
+		FREETZ_SHOW_DEVELOPER \
+		) && ( \
+		FREETZ_TYPE_1200 || \
+		FREETZ_TYPE_1209 || \
+		FREETZ_TYPE_1750 || \
+		FREETZ_TYPE_1759 || \
+		FREETZ_TYPE_2400 || \
+		FREETZ_TYPE_6000 || \
+		FREETZ_SHOW_DEVELOPER \
+		)
+	default n
+	help
+		Enables hidden menu for internet and certificate settings
+		in AVM user web interface for wifi extenders.
+		This allows the upload of custom user certificates instead
+		of box's self signed  certificates for secure https traffic.
+		Tested for repeaters 1200AX, 1750E, 2400 and 6000 with Firmware > 7.2X.
+		Developer can activate this patch at own risk also
+		for other (untested) repeaters and firmwares.
 
 config FREETZ_PATCH_SIGNED
 	bool "Patch web menu signed message"
@@ -1347,8 +1347,13 @@
 
 config FREETZ_ENABLE_GPON_SERIAL
 	bool "Enable GPON serial textbox"
-	depends on FREETZ_TYPE_FIBER && FREETZ_AVM_VERSION_07_5X_MAX
+	depends on FREETZ_AVM_VERSION_07_5X_MIN && !FREETZ_TYPE_FIRMWARE_INHAUS
 	depends on !FREETZ_REMOVE_SUPPORT
+	depends on \
+		( FREETZ_AVM_VERSION_07_5X_MIN && ( \
+			FREETZ_TYPE_5530 || \
+			FREETZ_TYPE_5590 \
+		))
 	default n
 	help
 		Re-enables access to the GPON serial textbox on support page.
@@ -1373,7 +1398,7 @@
 
 menuconfig FREETZ_PATCH_MODFS_BOOT_MANAGER
 	bool "Add yf-bootmanager"
-	depends on !FREETZ_ADD_INACTIVE_SYSTEM || (FREETZ_ADD_INACTIVE_SYSTEM && FREETZ_AVM_PROP_INNER_FILESYSTEM_TYPE_CPIO)
+	depends on !FREETZ_ADD_INACTIVE_SYSTEM || (FREETZ_ADD_INACTIVE_SYSTEM && FREETZ_AVM_PROP_INNER_FILESYSTEM_TYPE_CPIO || FREETZ_PACKAGE_DTC_fitdump)
 	depends on FREETZ_AVM_VERSION_06_5X_MIN
 	depends on ( FREETZ_AVM_HAS_SEPARATE_FILESYSTEM_IMAGE && !FREETZ_AVM_HAS_FWLAYOUT_5 ) || \
 		                   ( FREETZ_AVM_HAS_FWLAYOUT_5 && (FREETZ_AVM_HAS_BLK_DEV_LOOP_BUILTIN || FREETZ_REPLACE_MODULE_AVAILABLE) )
--- patches/scripts/823-show_internet_extender.sh	(nonexistent)
+++ patches/scripts/823-show_internet_extender.sh	(working copy)
@@ -0,0 +1,9 @@
+[ "$FREETZ_PATCH_EXTENDER_INTERNET" == "y" ] || return 0
+[ "$FREETZ_TYPE_EXTENDER" == "y" ] || return 0
+echo1 "enabling internet and certificate config for wifi extender"
+
+# patcht Internet > Freigaben
+
+modsed \
+  's/\(.*\)\(\["meshSet"\] = {.*\)/\1\["remoteHttps"\] = config\.REMOTE_HTTPS and {\n\["show"\] = true,\n\["lua"\] = "internet\/remote_https\.lua",\n\["help"\] = \(forLuaOnly and "hilfe_internet_remote_https"\) or true\n} or nil,\n\2/' \
+  "${MENU_DATA_LUA}"
