--- make/pkgs/mod/files/root/etc/default.mod/mod.cfg
+++ make/pkgs/mod/files/root/etc/default.mod/mod.cfg	2025-05-23 06:44:05.825844284 +0200
@@ -37,7 +37,7 @@
 export MOD_SHUTDOWN_LAST='cifsmount davfs2 autofs'
 export MOD_PATH_WTMP='/var/log/'
 export MOD_GET_IP_METHOD='--all'
-export MOD_GET_IP_STUN='stun.1und1.de'
+export MOD_GET_IP_STUN=''
 export MOD_SKIN='cuma'
 export MOD_LANG='en'
 export MOD_RESOLV_DNS='127.0.0.1'
