--- config/ui/firmware.in
+++ config/ui/firmware.in	2024-09-18 15:02:07.869804716 +0200
@@ -704,6 +704,7 @@
 
 choice
 	prompt "Firmware version"
+	default FREETZ_TYPE_FIRMWARE_08_0X
 	default FREETZ_TYPE_FIRMWARE_07_8X
 	default FREETZ_TYPE_FIRMWARE_07_5X
 	default FREETZ_TYPE_FIRMWARE_07_2X
@@ -777,7 +778,21 @@
 		bool "FRITZ!OS 07.50+"
 		select FREETZ_AVM_VERSION_07_5X
 		depends on FREETZ_AVM_HAS_FIRMWARE_07_5X
+		
+	config FREETZ_TYPE_FIRMWARE_07_5X_LABOR
+		bool "FRITZ!OS 07.5x Labor - UNTESTED"
+		select FREETZ_AVM_VERSION_07_5X
+		depends on FREETZ_AVM_HAS_FIRMWARE_07_5X_LABOR && FREETZ_SHOW_EXPERT
+		help
+			Enable this to use an AVM Labor firmware as base for modding.
 
+	config FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
+		bool "FRITZ!OS 07.5x Inhaus - UNTESTED"
+		select FREETZ_AVM_VERSION_07_5X
+#		select FREETZ_TYPE_FIRMWARE_DETECT_LATEST if FREETZ_AVM_HAS_FIRMWARE_07_5X_LABOR
+		depends on FREETZ_AVM_HAS_FIRMWARE_07_5X_INHAUS && FREETZ_SHOW_EXPERT
+		help
+			Enable this to use an AVM Inhaus firmware as base for modding.
 
 	config FREETZ_TYPE_FIRMWARE_07_8X
 		bool "FRITZ!OS 07.80+"
@@ -785,13 +800,10 @@
 		depends on FREETZ_AVM_HAS_FIRMWARE_07_8X
 
 
-	config FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
-		bool "FRITZ!OS 07.90 Inhaus - UNTESTED"
+	config FREETZ_TYPE_FIRMWARE_08_0X
+		bool "FRITZ!OS 08.00+"
 		select FREETZ_AVM_VERSION_08_0X
-		select FREETZ_TYPE_FIRMWARE_DETECT_LATEST if FREETZ_AVM_HAS_FIRMWARE_08_0X_LABOR
-		depends on FREETZ_AVM_HAS_FIRMWARE_08_0X_INHAUS && FREETZ_SHOW_EXPERT
-		help
-			Enable this to use an AVM Inhaus firmware as base for modding.
+		depends on FREETZ_AVM_HAS_FIRMWARE_08_0X
 
 	config FREETZ_TYPE_FIRMWARE_08_0X_LABOR
 		bool "FRITZ!OS 07.90 Labor - UNTESTED"
@@ -801,10 +813,13 @@
 			Enable this to use an AVM Labor firmware as base for modding.
 			For more details see <http://www.avm.de/Labor/>.
 
-	config FREETZ_TYPE_FIRMWARE_08_0X
-		bool "FRITZ!OS 08.00+"
+	config FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
+		bool "FRITZ!OS 07.90 Inhaus - UNTESTED"
 		select FREETZ_AVM_VERSION_08_0X
-		depends on FREETZ_AVM_HAS_FIRMWARE_08_0X
+#		select FREETZ_TYPE_FIRMWARE_DETECT_LATEST if FREETZ_AVM_HAS_FIRMWARE_08_0X_LABOR
+		depends on FREETZ_AVM_HAS_FIRMWARE_08_0X_INHAUS && FREETZ_SHOW_EXPERT
+		help
+			Enable this to use an AVM Inhaus firmware as base for modding.
 
 
 endchoice # "Firmware version" #
@@ -874,6 +889,7 @@
 	bool
 	default y
 	depends on \
+		FREETZ_TYPE_FIRMWARE_07_5X_INHAUS || \
 		FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 
 config FREETZ_TYPE_FIRMWARE_LABOR
