--- config/ui/firmware.in	2024-03-15 16:55:21.108639796 +0100
+++ config/ui/firmware.in	2024-03-15 17:44:38.088248856 +0100
@@ -729,6 +729,30 @@
 		bool "FRITZ!OS 07.50+"
 		select FREETZ_AVM_VERSION_07_5X
 		depends on FREETZ_AVM_HAS_FIRMWARE_07_5X
+ 
+	config FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+		bool "FRITZ!OS 07.50+ LaborPlus - UNTESTED"
+ 		select FREETZ_AVM_VERSION_07_5X
+		depends on FREETZ_AVM_HAS_FIRMWARE_07_5X_PLUS && FREETZ_SHOW_EXPERT
+ 		help
+ 			Enable this to use an AVM Labor firmware as base for modding.
+ 			For more details see <https://service.avm.de/downloads/LaborPlus/>.
+ 			
+	config FREETZ_TYPE_FIRMWARE_07_5X_LABOR
+		bool "FRITZ!OS 07.50+ Labor - UNTESTED"
+		select FREETZ_AVM_VERSION_07_5X
+		depends on FREETZ_AVM_HAS_FIRMWARE_07_5X_LABOR && FREETZ_SHOW_EXPERT
+		help
+			Enable this to use an AVM Labor firmware as base for modding.
+			For more details see <http://www.avm.de/Labor/>.
+
+	config FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
+		bool "FRITZ!OS 07.50+ Inhaus - UNTESTED"
+		select FREETZ_AVM_VERSION_07_5X
+#		select FREETZ_TYPE_FIRMWARE_DETECT_LATEST
+		depends on FREETZ_AVM_HAS_FIRMWARE_07_5X_INHAUS && FREETZ_SHOW_EXPERT
+		help
+			Enable this to use an AVM Inhaus firmware as base for modding.
 
 
 	config FREETZ_TYPE_FIRMWARE_07_8X
@@ -737,21 +761,37 @@
 		depends on FREETZ_AVM_HAS_FIRMWARE_07_8X
 
 	config FREETZ_TYPE_FIRMWARE_07_8X_LABOR
-		bool "FRITZ!OS 07.70 Labor - UNTESTED"
+		bool "FRITZ!OS 07.70+ Labor - UNTESTED"
 		select FREETZ_AVM_VERSION_07_8X
 		depends on FREETZ_AVM_HAS_FIRMWARE_07_8X_LABOR && FREETZ_SHOW_EXPERT
 		help
 			Enable this to use an AVM Labor firmware as base for modding.
 			For more details see <http://www.avm.de/Labor/>.
-
+			
 	config FREETZ_TYPE_FIRMWARE_07_8X_INHAUS
-		bool "FRITZ!OS 07.70 Inhaus - UNTESTED"
+		bool "FRITZ!OS 07.70+ Inhaus - UNTESTED"
 		select FREETZ_AVM_VERSION_07_8X
-		select FREETZ_TYPE_FIRMWARE_DETECT_LATEST
+#		select FREETZ_TYPE_FIRMWARE_DETECT_LATEST
 		depends on FREETZ_AVM_HAS_FIRMWARE_07_8X_INHAUS && FREETZ_SHOW_EXPERT
 		help
 			Enable this to use an AVM Inhaus firmware as base for modding.
 
+	config FREETZ_TYPE_FIRMWARE_07_8X_PLUS
+		bool "FRITZ!OS 07.80+ LaborPlus - UNTESTED"
+ 		select FREETZ_AVM_VERSION_07_8X
+		depends on FREETZ_AVM_HAS_FIRMWARE_07_8X_PLUS && FREETZ_SHOW_EXPERT
+ 		help
+ 			Enable this to use an AVM Labor firmware as base for modding.
+ 			For more details see <https://service.avm.de/downloads/LaborPlus/>.
+ 			
+	config FREETZ_TYPE_FIRMWARE_07_9X_INHAUS
+		bool "FRITZ!OS 07.90+ Inhaus - UNTESTED"
+		select FREETZ_AVM_VERSION_07_9X
+#		select FREETZ_TYPE_FIRMWARE_DETECT_LATEST
+		depends on FREETZ_AVM_HAS_FIRMWARE_07_9X_INHAUS && FREETZ_SHOW_EXPERT
+		help
+			Enable this to use an AVM Inhaus firmware as base for modding.
+
 
 endchoice # "Firmware version" #
 
@@ -807,25 +847,44 @@
 	default y
 	depends on \
 		!FREETZ_TYPE_FIRMWARE_LABOR && \
+		!FREETZ_TYPE_FIRMWARE_PLUS && \
 		!FREETZ_TYPE_FIRMWARE_INHAUS
 
 config FREETZ_AVM_HAS_FIRMWARE_LABOR
 	bool
 	default y
 	depends on \
+		FREETZ_AVM_HAS_FIRMWARE_07_5X_LABOR || \
 		FREETZ_AVM_HAS_FIRMWARE_07_8X_LABOR
 
+config FREETZ_AVM_HAS_FIRMWARE_PLUS
+	bool
+	default y
+	depends on \
+		FREETZ_AVM_HAS_FIRMWARE_07_5X_PLUS || \
+		FREETZ_AVM_HAS_FIRMWARE_07_8X_PLUS
+
 config FREETZ_TYPE_FIRMWARE_INHAUS
 	bool
 	default y
 	depends on \
 		FREETZ_TYPE_FIRMWARE_07_1X_INHAUS || \
-		FREETZ_TYPE_FIRMWARE_07_8X_INHAUS
+		FREETZ_TYPE_FIRMWARE_07_5X_INHAUS || \
+		FREETZ_TYPE_FIRMWARE_07_8X_INHAUS || \
+		FREETZ_TYPE_FIRMWARE_07_9X_INHAUS
+
+config FREETZ_TYPE_FIRMWARE_PLUS
+	bool
+	default y
+	depends on \
+		FREETZ_TYPE_FIRMWARE_07_5X_PLUS || \
+		FREETZ_TYPE_FIRMWARE_07_8X_PLUS
 
 config FREETZ_TYPE_FIRMWARE_LABOR
 	bool
 	default y
 	depends on \
+		FREETZ_TYPE_FIRMWARE_07_8X_LABOR || \
 		FREETZ_TYPE_FIRMWARE_07_8X_LABOR
 
 ###
