--- make/kernel/configs/freetz/config-puma6.x86-6590_07.02	2023-06-17 12:27:48.712257056 +0200
+++ make/kernel/configs/freetz/config-puma6.x86-6590_07.02	2023-06-18 15:52:09.504314880 +0200
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/x86 2.6.39.3 Kernel Configuration
-# Sun Mar 10 14:46:44 2019
+# Sun Jun 18 15:52:08 2023
 #
 # CONFIG_64BIT is not set
 CONFIG_X86_32=y
@@ -1311,8 +1311,18 @@
 #
 # CONFIG_FDDI is not set
 # CONFIG_HIPPI is not set
-# CONFIG_PPP is not set
+CONFIG_PPP=m
+# CONFIG_PPP_MULTILINK is not set
+# CONFIG_PPP_FILTER is not set
+CONFIG_PPP_ASYNC=m
+# CONFIG_PPP_SYNC_TTY is not set
+CONFIG_PPP_DEFLATE=m
+# CONFIG_PPP_BSDCOMP is not set
+CONFIG_PPP_MPPE=m
+# CONFIG_PPPOE is not set
+# CONFIG_PPPOL2TP is not set
 # CONFIG_SLIP is not set
+CONFIG_SLHC=m
 # CONFIG_NET_FC is not set
 # CONFIG_NETCONSOLE is not set
 # CONFIG_NETPOLL is not set
@@ -2443,16 +2453,20 @@
 # CONFIG_NFS_V4 is not set
 CONFIG_NFSD=m
 # CONFIG_NFSD_DEPRECATED is not set
+CONFIG_NFSD_V2_ACL=y
 CONFIG_NFSD_V3=y
-# CONFIG_NFSD_V3_ACL is not set
-# CONFIG_NFSD_V4 is not set
+CONFIG_NFSD_V3_ACL=y
+CONFIG_NFSD_V4=y
 CONFIG_LOCKD=m
 CONFIG_LOCKD_V4=y
+CONFIG_NFS_ACL_SUPPORT=m
 CONFIG_NFS_COMMON=y
 CONFIG_SUNRPC=m
+CONFIG_SUNRPC_GSS=m
 # CONFIG_CEPH_FS is not set
 CONFIG_CIFS=m
-# CONFIG_CIFS_STATS is not set
+CONFIG_CIFS_STATS=y
+# CONFIG_CIFS_STATS2 is not set
 # CONFIG_CIFS_WEAK_PW_HASH is not set
 # CONFIG_CIFS_UPCALL is not set
 # CONFIG_CIFS_XATTR is not set
@@ -2460,7 +2474,7 @@
 # CONFIG_CIFS_DFS_UPCALL is not set
 # CONFIG_CIFS_EXPERIMENTAL is not set
 # CONFIG_NCP_FS is not set
-CONFIG_CODA_FS=m
+# CONFIG_CODA_FS is not set
 # CONFIG_AFS_FS is not set
 
 #
