--- make/toolchain/download-toolchain.mk	2022-07-06 02:38:20.928262426 +0200
+++ make/toolchain/download-toolchain.mk	2022-07-07 08:50:48.599156415 +0200
@@ -3,7 +3,7 @@
 include $(TOOLCHAIN_DIR)/make/target/libtool-staging/libtool-staging.mk
 include $(TOOLCHAIN_DIR)/make/target/uclibc/uclibc.mk
 
-ifeq ($(strip $(FREETZ_TOOLCHAIN_CCACHE)),y)
+ifeq ($(strip $(FREETZ_TOOLCHAIN_CCACHE)),n)
 	CCACHE:=ccache-kernel ccache
 endif
 
