--- make/libs/freetype/freetype.mk	2022-08-11 17:31:33.555017000 +0200
+++ make/libs/freetype/freetype.mk	2022-08-12 07:05:32.487502455 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 2.10.1)
-$(PKG)_LIB_VERSION:=6.17.1
+$(call PKG_INIT_LIB, 2.12.1)
+$(PKG)_LIB_VERSION:=6
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=16dbfa488a21fe827dc27eaf708f42f7aa3bb997d745d31a19781628c36ba26f
+$(PKG)_HASH:=4766f20157cc4cf0cd292f80bf917f92d1c439b243ac3018debf6b9140c41a7f
 $(PKG)_SITE:=@SF/freetype
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/objs/.libs/libfreetype.so.$($(PKG)_LIB_VERSION)
