--- make/libs/jpeg/jpeg.mk	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/jpeg/jpeg.mk	2022-09-24 09:26:10.757851640 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 9d)
-$(PKG)_LIB_VERSION:=9.4.0
+$(call PKG_INIT_LIB, 9e)
+$(PKG)_LIB_VERSION:=9
 $(PKG)_SOURCE:=jpegsrc.v$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=6c434a3be59f8f62425b2e3c077e785c9ce30ee5874ea1c270e843f273ba71ee
+$(PKG)_HASH:=4077d6a6a75aeb01884f708919d25934c93305e49f7e3f36db9129320e6f4f3d
 $(PKG)_SITE:=http://ijg.org/files
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/.libs/libjpeg.so.$($(PKG)_LIB_VERSION)
--- make/libs/jpeg/external.in	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/jpeg/external.in	2022-09-24 09:27:45.155536259 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libjpeg.so.9.4.0
+		 ${FREETZ_LIBRARY_DIR}/libjpeg.so.9
 
--- make/libs/jpeg/external.files	2022-09-22 19:57:32.295197000 +0200
+++ make/libs/jpeg/external.files	2022-09-24 09:27:55.467292907 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libjpeg" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libjpeg.so.9.4.0"
+[ "$EXTERNAL_FREETZ_LIB_libjpeg" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libjpeg.so.9"
