--- make/libs/libcap/libcap.mk	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/libcap/libcap.mk	2022-09-29 17:07:16.519005435 +0200
@@ -1,10 +1,10 @@
-$(call PKG_INIT_LIB, $(if $(FREETZ_AVM_GCC_4_MAX),2.49,2.65))
+$(call PKG_INIT_LIB, $(if $(FREETZ_AVM_GCC_4_MAX),2.49,2.66))
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
 $(PKG)_HASH_ABANDON:=e98bc4d93645082ec787730b0fd1a712b38882465c505777de17c338831ee181
-$(PKG)_HASH_CURRENT:=73e350020cc31fe15360879d19384ffa3395a825f065fcf6bda3a5cdf965bebd
+$(PKG)_HASH_CURRENT:=15c40ededb3003d70a283fe587a36b7d19c8b3b554e33f86129c059a4bb466b2
 $(PKG)_HASH:=$($(PKG)_HASH_$(if $(FREETZ_AVM_GCC_4_MAX),ABANDON,CURRENT))
 $(PKG)_SITE:=@KERNEL/linux/libs/security/linux-privs/libcap2
-### VERSION:=2.49/2.65
+### VERSION:=2.49/2.66
 ### WEBSITE:=https://sites.google.com/site/fullycapable/
 ### MANPAGE:=https://pkg.go.dev/kernel.org/pub/linux/libs/security/libcap/cap
 ### CHANGES:=https://sites.google.com/site/fullycapable/release-notes-for-libcap
--- make/libs/libcap/external.files	2022-09-24 19:44:27.159340000 +0200
+++ make/libs/libcap/external.files	2022-09-29 17:02:57.031597214 +0200
@@ -1,3 +1,3 @@
-[ "FREETZ_AVM_GCC_4_MAX" == "y" ] && ELIBVER="2.49" || ELIBVER="2.65"
+[ "FREETZ_AVM_GCC_4_MAX" == "y" ] && ELIBVER="2.49" || ELIBVER="2.66"
 
 [ "$EXTERNAL_FREETZ_LIB_libcap" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libcap.so.$ELIBVER"
