--- make/libs/libidn/libidn.mk	2022-10-02 09:28:13.079561000 +0200
+++ make/libs/libidn/libidn.mk	2022-10-02 10:07:01.620903228 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 1.38)
-$(PKG)_LIB_VERSION:=12.6.3
+$(call PKG_INIT_LIB, 1.9)
+$(PKG)_LIB_VERSION:=11.5.38
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=de00b840f757cd3bb14dd9a20d5936473235ddcba06d4bc2da804654b8bbf0f6
+$(PKG)_HASH:=bd5d01e9e8b2d78b56828e7533a394239146cdca3965b4dd891508cbeb727470
 $(PKG)_SITE:=@GNU/$(pkg)
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/$(pkg).so.$($(PKG)_LIB_VERSION)
