--- tools/freetz_download
+++ tools/freetz_download	2026-03-11 10:53:34.495651562 +0100
@@ -344,6 +345,8 @@
 				done
 			elif [ ${url:0:4} == "@AVM" ]; then
 				sites[${#sites[@]}]=@DLTOKEN/https://api.github.com/repos/Freetz-NG/internal/releases/tags/firmwares
+				sites[${#sites[@]}]=http://192.168.178.9/fritzbox/download.avm.de/fritzbox/$subpath
+				sites[${#sites[@]}]=http://192.168.178.9/fritzbox/download.avm.de/$subpath
 				sites[${#sites[@]}]=https://download.avm.de/fritzbox/$subpath
 				sites[${#sites[@]}]=https://download.avm.de/$subpath
 #				sites[${#sites[@]}]=https://download.avm.de/fritzwlan/$subpath
