--- tools/freetz_download
+++ tools/freetz_download	2024-12-23 17:13:27.634093931 +0100
@@ -296,11 +299,13 @@
 #				sites[(( num_sites++ ))]=https://download.avm.de/fritzdect/$subpath
 				sites[(( num_sites++ ))]=https://avm.de/$subpath
 				sites[(( num_sites++ ))]=https://avm.de/fileadmin/user_upload/DE/Labor/Download
+				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/Smart24P1NL1/$subpath
 				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/Smart24P1FCS/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/Smart24P1/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/GoFiber23NL1/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/GoFiber23/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/FCS23/$subpath
+				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/MOVE21NL2/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/MOVE21NL1/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/MOVE21/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/WU22PSQ19/$subpath
