--- tools/freetz_download
+++ tools/freetz_download
@@ -289,6 +290,8 @@
 				done
 			elif [ ${url:0:4} == "@AVM" ]; then
 				subpath=${url/@AVM\//}
+				sites[(( num_sites++ ))]=http://192.168.178.9/fritzbox/download.avm.de/fritzbox/$subpath
+				sites[(( num_sites++ ))]=http://192.168.178.9/fritzbox/download.avm.de/$subpath
 				sites[(( num_sites++ ))]=https://download.avm.de/fritzbox/$subpath
 				sites[(( num_sites++ ))]=https://download.avm.de/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/fritzwlan/$subpath
@@ -304,7 +307,8 @@
 				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/GoFiber23NL1/$subpath
 				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/GoFiber23/$subpath
 				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/FCS23/$subpath
-#				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/MOVE21NL1/$subpath
+				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/MOVE21NL2/$subpath
+				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/MOVE21NL1/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/MOVE21/$subpath
 #				sites[(( num_sites++ ))]=https://download.avm.de/inhaus/WU22PSQ19/$subpath
 				sites[(( num_sites++ ))]=https://download.avm.de/testfeld/$subpath
