--- config/mod/dl-firmware.in	2022-10-10 19:46:55.710515000 +0200
+++ config/mod/dl-firmware.in	2022-10-11 20:32:21.686465207 +0200
@@ -796,7 +796,7 @@
 	default "FRITZ.Box_7340.int.99.05.23.image"                                     if FREETZ_TYPE_7340_7390 &&  FREETZ_TYPE_FIRMWARE_05_2X
 	default "FRITZ.Box_Fon_WLAN_7340.en-de-es-it-fr.99.06.06.image"                 if FREETZ_TYPE_7340_7390 &&  FREETZ_TYPE_FIRMWARE_06_0X
 	#
-	default "FRITZ.Box_7412-06.86.image"                                            if FREETZ_TYPE_7412_7430
+	default "FRITZ.Box_7412.137.06.87.image"                                        if FREETZ_TYPE_7412_7430
 	#
 	default "FRITZ.Box_Fon_WLAN_7570_vDSL.en-de-fr.75.04.92.image"                  if FREETZ_TYPE_7570_7270
 	#
