--- make/libs/libosip2/libosip2.mk	2022-10-02 09:28:13.083561000 +0200
+++ make/libs/libosip2/libosip2.mk	2022-10-03 03:47:00.361547997 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 3.5.0)
-$(PKG)_LIB_VERSION:=6.2.0
+$(call PKG_INIT_LIB, 5.3.1)
+$(PKG)_LIB_VERSION:=15.0.1
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=dd955daa24d9ce2de6709b8c13e7c04ebc3afa8ac094d6a15a02a075be719a91
+$(PKG)_HASH:=fe82fe841608266ac15a5c1118216da00c554d5006e2875a8ac3752b1e6adc79
 $(PKG)_SITE:=@GNU/osip
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/src/osip2/.libs/libosip2.so.$($(PKG)_LIB_VERSION)
--- make/libs/libosip2/external.in	2022-10-02 09:28:13.083561000 +0200
+++ make/libs/libosip2/external.in	2022-10-03 03:49:08.931596886 +0200
@@ -4,7 +4,7 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libosip2.so.6.2.0
+		 ${FREETZ_LIBRARY_DIR}/libosip2.so.15.0.1
 
 config EXTERNAL_FREETZ_LIB_libosipparser2
 	depends on EXTERNAL_ENABLED && FREETZ_LIB_libosipparser2
@@ -12,5 +12,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libosipparser2.so.6.2.0
+		 ${FREETZ_LIBRARY_DIR}/libosipparser2.so.15.0.1
 
--- make/libs/libosip2/external.files	2022-10-02 09:28:13.083561000 +0200
+++ make/libs/libosip2/external.files	2022-10-03 03:48:44.699960973 +0200
@@ -1,2 +1,2 @@
-[ "$EXTERNAL_FREETZ_LIB_libosip2" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libosip2.so.6.2.0"
-[ "$EXTERNAL_FREETZ_LIB_libosipparser2" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libosipparser2.so.6.2.0"
+[ "$EXTERNAL_FREETZ_LIB_libosip2" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libosip2.so.15.0.1"
+[ "$EXTERNAL_FREETZ_LIB_libosipparser2" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libosipparser2.so.15.0.1"
