--- patches/scripts/832-hide_dsl_overview-hint.sh	2023-10-19 05:18:48.695298000 +0200
+++ patches/scripts/832-hide_dsl_overview-hint.sh	2023-11-03 22:59:50.225525765 +0100
@@ -1,7 +1,9 @@
 [ "$FREETZ_MODIFY_DSL_WARNING" == "y" ] || return 0
 echo1 "patching dsl overview"
 
-if [ "$FREETZ_AVM_VERSION_07_2X_MIN" == "y" ] ; then
+if [ "$FREETZ_AVM_VERSION_07_8X_MIN" == "y" ] ; then
+	PVER="07_8X"
+elif [ "$FREETZ_AVM_VERSION_07_2X_MIN" == "y" ] ; then
 	PVER="07_2X"
 elif [ "$FREETZ_AVM_VERSION_07_0X_MIN" == "y" ] ; then
 	PVER="07_0X"
