--- config/mod/dl-firmware.in	2022-10-10 19:46:55.710515000 +0200
+++ config/mod/dl-firmware.in	2022-10-11 20:32:21.686465207 +0200
@@ -99,7 +99,8 @@
 	#
 	default "@AVM/fritzbox-5530-fiber/{other,deutschland}/fritz.os,@AVM/5530Fiber"                                     if FREETZ_TYPE_5530
 	#
-	default "@AVM/fritzbox-5590-fiber/{other,deutschland}/fritz.os"                                                    if FREETZ_TYPE_5590
+	default "@AVM/fritzbox-5590-fiber/{other,deutschland}/fritz.os,@AVM/5530Fiber"                                     if FREETZ_TYPE_5590
+	
 	#
 	default "@AVM/fritzwlan/fritzrepeater-6000/deutschland/fritz.os"                                                   if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_25
 	#
@@ -300,7 +301,7 @@
 	default "FRITZ.Powerline_1260E-07.21-Release.image"                                    if FREETZ_TYPE_1260_V1 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Powerline_1260E.157.07.29.image"                                        if FREETZ_TYPE_1260_V1 && FREETZ_TYPE_FIRMWARE_07_25
 	#
-	default "FRITZ.Powerline_1260.249.07.29-Release.image"                                 if FREETZ_TYPE_1260_V2 && FREETZ_TYPE_FIRMWARE_07_25
+	default "FRITZ.Powerline_1260.249.07.29.image"		                               if FREETZ_TYPE_1260_V2 && FREETZ_TYPE_FIRMWARE_07_25
 	default "FRITZ.Powerline_1260-07.39-101040-Inhaus.image"                               if FREETZ_TYPE_1260_V2 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_WLAN_Repeater_1750E.134.06.32.image"                                if (FREETZ_TYPE_1750 || FREETZ_TYPE_1759_1750) && FREETZ_TYPE_FIRMWARE_06_2X
@@ -408,6 +409,7 @@
 	default "FRITZ.Box_5530_Fiber-07.39-102667.image"                                      if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_5X_LABOR
 	#
 	default "FRITZ.Box_5590-07.30.image"                                                   if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_25
+	default "FRITZ.Box_5590_Fiber-07.39-102864.image"                                      if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_5X_LABOR
 	#
 	default "FRITZ.Repeater_6000-07.30.image"                                              if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_25
 	default "FRITZ.Repeater_6000-07.39-101161-Labor.image"                                 if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_5X_LABOR
@@ -716,6 +718,7 @@
 	default "fritzbox-4040-labor-101165.zip"          if FREETZ_TYPE_4040
 	default "fritzbox-4060-labor-101164.zip"          if FREETZ_TYPE_4060
 	default "fritzbox-5530-fiber-labor-102667.zip"    if FREETZ_TYPE_5530
+	default "fritzbox-5590-fiber-labor-102864.zip"    if FREETZ_TYPE_5530
 	default "fritz-repeater-6000-labor-101161.zip"    if FREETZ_TYPE_6000
 	default "fritzbox-6490-labor-101216.zip"          if FREETZ_TYPE_6490
 	default "fritzbox-6590-labor-101217.zip"          if FREETZ_TYPE_6590
