--- config/mod/dl-firmware.in	2024-02-07 11:33:43.309888992 +0100
+++ config/mod/dl-firmware.in	2024-02-07 19:19:47.689967581 +0100
@@ -31,7 +31,7 @@
 	#
 	default "@AVM/fritzwlan/fritzrepeater-1200/{other,deutschland}/fritz.os,@AVM/1200"                                 if FREETZ_TYPE_1200
 	#
-	default "@AVM/fritzwlan/fritzrepeater-1200-ax/{other,deutschland}/fritz.os,@AVM/firmware/1200AX"                   if FREETZ_TYPE_1209
+	default "@AVM/fritzwlan/fritzrepeater-1200-ax/{other,deutschland}/fritz.os,@AVM/1200AX"                            if FREETZ_TYPE_1209
 	#
 	default "@AVM/fritzpowerline/fritzpowerline-1240e/{other,deutschland}/fritz.os"                                    if FREETZ_TYPE_1240
 	#
@@ -51,7 +51,7 @@
 	#
 	default "@AVM/fritzwlan/fritzrepeater-3000/{other,deutschland}/fritz.os,@AVM/3000"                                 if FREETZ_TYPE_3000
 	#
-	default "@AVM/fritzwlan/fritzrepeater-3000-ax/{other,deutschland}/fritz.os"                                        if FREETZ_TYPE_3009
+	default "@AVM/fritzwlan/fritzrepeater-3000-ax/{other,deutschland}/fritz.os,@AVM/3000AX"                            if FREETZ_TYPE_3009
 	#
 	default "@AVM/fritzbox.sl_wlan/firmware"                                                                           if FREETZ_TYPE_3020
 	#
@@ -104,7 +104,7 @@
 	#
 	default "@AVM/fritzbox-5590-fiber/{other,deutschland}/fritz.os,@AVM/5590Fiber"                                     if FREETZ_TYPE_5590
 	#
-	default "@AVM/fritzwlan/fritzrepeater-6000/deutschland/fritz.os"                                                   if FREETZ_TYPE_6000
+	default "@AVM/fritzwlan/fritzrepeater-6000/deutschland/fritz.os,@AVM/6000"                                         if FREETZ_TYPE_6000
 	#
 	default "@AVM/firmware/6430/65797387"                                                                              if FREETZ_TYPE_6430 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "@AVM/fritzbox-6430-cable/deutschland/fritz.os"                                                            if FREETZ_TYPE_6430
@@ -123,9 +123,9 @@
 	default "@AVM/fritzbox-6591-cable/{deutschland,other}/fritz.os,@AVM/6591Cable"                                     if FREETZ_TYPE_6591
 	#
 	default "@AVM/firmware/6660/89746785"                                                                              if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_1X
-	default "@AVM/fritzbox-6660-cable/{deutschland,other}/fritz.os"                                                    if FREETZ_TYPE_6660
+	default "@AVM/fritzbox-6660-cable/{deutschland,other}/fritz.os,@AVM/6660Cable"                                     if FREETZ_TYPE_6660
 	#
-	default "@AVM/fritzbox-6690-cable/{deutschland,other}/fritz.os,@AVM/FRITZ%21Box%206690%20Cable/other/fritz.os"     if FREETZ_TYPE_6690
+	default "@AVM/fritzbox-6690-cable/{deutschland,other}/fritz.os,@AVM/6690Cable"                                     if FREETZ_TYPE_6690
 	#
 	default "@AVM/{fritzbox-6810-lte/deutschland/fritz.os,fritzbox_6810_lte/firmware/deutsch}"                         if FREETZ_TYPE_6810 && FREETZ_TYPE_LANG_DE
 	default "@AVM/{fritzbox-6810-lte/other/fritz.os,fritzbox_6810_lte/firmware/english}"                               if FREETZ_TYPE_6810 && FREETZ_TYPE_LANG_EN
@@ -287,9 +287,11 @@
 	default "FRITZ.Repeater_1200-07.14.image"                                              if FREETZ_TYPE_1200 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Repeater_1200-07.29.image"                                              if FREETZ_TYPE_1200 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_1200-07.57.image"                                              if FREETZ_TYPE_1200 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Repeater_1200-07.57-110997-Inhaus.image"                                if FREETZ_TYPE_1200 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Repeater_1200_AX-07.31.image"                                           if FREETZ_TYPE_1209 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_1200_AX-07.57.image"                                           if FREETZ_TYPE_1209 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Repeater_1200_AX-07.57-111007-Inhaus.image"                             if FREETZ_TYPE_1209 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Powerline_1240E.06.92.image"                                            if FREETZ_TYPE_1240 && FREETZ_TYPE_FIRMWARE_06_9X
 	default "FRITZ.Powerline_1240E.150.07.01.image"                                        if FREETZ_TYPE_1240 && FREETZ_TYPE_FIRMWARE_07_0X
@@ -321,6 +323,7 @@
 	default "FRITZ.Repeater_2400-07.12.image"                                              if FREETZ_TYPE_2400 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Repeater_2400-07.29.image"                                              if FREETZ_TYPE_2400 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_2400-07.57.image"                                              if FREETZ_TYPE_2400 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Repeater_2400-07.57-110995-Inhaus.image"                                if FREETZ_TYPE_2400 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "fritz.box_sl_wlan.09.04.34.image"                                             if FREETZ_TYPE_3020
 	#
@@ -328,9 +331,11 @@
 	default "FRITZ.Repeater_3000-07.14.image"                                              if FREETZ_TYPE_3000 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Repeater_3000-07.29.image"                                              if FREETZ_TYPE_3000 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_3000-07.57.image"                                              if FREETZ_TYPE_3000 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Repeater_3000-07.57-110999-Inhaus.image"                                if FREETZ_TYPE_3000 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Repeater_3000_AX-07.42.image"                                           if FREETZ_TYPE_3009 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_3000_AX-07.57-V2.image/FRITZ.Repeater_3000_AX-07.57.image"     if FREETZ_TYPE_3009 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Repeater_3000_AX-07.57-111005-Inhaus.image"                             if FREETZ_TYPE_3009 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "fritz.box_wlan_3030.21.04.34.image"                                           if FREETZ_TYPE_3030
 	#
@@ -385,6 +390,8 @@
 	#
 	default "FRITZ.Box_4060-07.30.image"                                                   if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_4060-07.57-V2.image/FRITZ.Box_4060-07.57.image"                     if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_4060-07.57-108832-LabPLUS.image"                                    if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+	default "FRITZ.Box_4060-07.57-109858-Inhaus.image"                                     if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "fritz.box_fon_5010.annexa.48.04.43.image"                                     if FREETZ_TYPE_5010
 	#
@@ -405,14 +412,20 @@
 	#
 	default "FRITZ.Box_5530-07.30.image"                                                   if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_5530_Fiber-07.58.image"                                             if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_5530_Fiber-07.70-110408-LabBETA.image"                              if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_8X_LABOR
+	default "FRITZ.Box_5530_Fiber-07.70-110390-Inhaus.image"                               if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_8X_INHAUS
 	default "FRITZ.Box_5530_Fiber-07.80.image"                                             if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_8X
 	#
 	default "FRITZ.Box_5590-07.30.image"                                                   if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_5590_Fiber-07.58-V2.image/FRITZ.Box_5590_Fiber-07.58.image"         if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_5590_Fiber-07.58-108561-LabPLUS.image"                              if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+	default "FRITZ.Box_5590_Fiber-07.70-110410-LabBETA.image"                              if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_8X_LABOR
+	default "FRITZ.Box_5590_Fiber-07.70-110392-Inhaus.image"                               if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_8X_INHAUS
 	default "FRITZ.Box_5590_Fiber-07.80.image"                                             if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_8X
 	#
 	default "FRITZ.Repeater_6000-07.31-V2.image/FRITZ.Repeater_6000-07.31.image"           if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_6000-07.57-V2.image/FRITZ.Repeater_6000-07.57.image"           if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Repeater_6000-07.57-111009-Inhaus.image"                                if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_6430_Cable-07.12.image"                                             if FREETZ_TYPE_6430 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_6430_Cable-07.30.image"                                             if FREETZ_TYPE_6430 && FREETZ_TYPE_FIRMWARE_07_2X
@@ -433,13 +446,18 @@
 	default "FRITZ.Box_6591_Cable-07.13.image"                                             if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_6591_Cable-07.29.image"                                             if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_6591_Cable-07.57.image"                                             if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_6591_Cable-07.57-110963-Inhaus.image"                               if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_6660_Cable-07.15.image"                                             if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_6660_Cable-07.29.image"                                             if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_6660_Cable-07.57.image"                                             if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_6660_Cable-07.57-108563-LabPLUS.image"                              if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+	default "FRITZ.Box_6660_Cable-07.57-110964-Inhaus.image"                               if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_6690_Cable-07.29.image"                                             if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_6690_Cable-07.57-V2.image/FRITZ.Box_6690_Cable-07.57.image"         if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_6690_Cable-07.57-108550-LabPLUS.image"                              if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+	default "FRITZ.Box_6690_Cable-07.57-110966-Inhaus.image"                               if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_6810_LTE.108.06.35.image"                                           if FREETZ_TYPE_6810 && FREETZ_TYPE_LANG_DE && FREETZ_TYPE_FIRMWARE_06_2X
 	default "FRITZ.Box_6810_LTE.en-de-es-it-fr.108.06.35.image"                            if FREETZ_TYPE_6810 && FREETZ_TYPE_LANG_EN && FREETZ_TYPE_FIRMWARE_06_2X
@@ -598,6 +616,7 @@
 	default "FRITZ.Box_7490-07.12.image"                                                   if FREETZ_TYPE_7490 && FREETZ_TYPE_LANG_DE && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_7490-07.29.image"                                                   if FREETZ_TYPE_7490 &&                        FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7490-07.57.image"                                                   if FREETZ_TYPE_7490 &&                        FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7490-07.57-109066-LabPLUS.image"                                    if FREETZ_TYPE_7490 &&                        FREETZ_TYPE_FIRMWARE_07_5X_PLUS
 	#
 	default "FRITZ.Box_7510-07.30.image"                                                   if FREETZ_TYPE_7510                        && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7510-07.57.image"                                                   if FREETZ_TYPE_7510                        && FREETZ_TYPE_FIRMWARE_07_5X
@@ -618,6 +637,8 @@
 	#
 	default "FRITZ.Box_7530_AX-07.31.image"                                                if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7530_AX-07.57-V2.image/FRITZ.Box_7530_AX-07.57.image"               if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7530_AX-07.70-110407-Labor.image"                                   if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_8X_LABOR
+	default "FRITZ.Box_7530_AX-07.70-110389-Inhaus.image"                                  if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_8X_INHAUS
 	default "FRITZ.Box_7530_AX-07.80.image"                                                if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_8X
 	#
 	default "FRITZ.Box_7560.en-de-es-it-fr-pl.149.06.83.image"                             if FREETZ_TYPE_7560 && FREETZ_TYPE_LANG_EN && FREETZ_TYPE_FIRMWARE_06_8X
@@ -647,6 +668,7 @@
 	default "FRITZ.Box_7583-07.12.image"                                                   if FREETZ_TYPE_7583 && FREETZ_TYPE_LANG_DE && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_7583-07.31.image"                                                   if FREETZ_TYPE_7583                        && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7583-07.57.image"                                                   if FREETZ_TYPE_7583                        && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7583-07.57-108834-LabPLUS.image"                                    if FREETZ_TYPE_7583                        && FREETZ_TYPE_FIRMWARE_07_5X_PLUS
 	#
 	default "FRITZ.Box_7583_VDSL-07.31.image"                                              if FREETZ_TYPE_7584                        && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7583_VDSL-07.57.image"                                              if FREETZ_TYPE_7584                        && FREETZ_TYPE_FIRMWARE_07_5X
@@ -660,12 +682,17 @@
 	default "FRITZ.Box_7590-07.12.image"                                                   if FREETZ_TYPE_7590 && FREETZ_TYPE_LANG_DE && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_7590-07.29.image"                                                   if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7590-07.57.image"                                                   if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7590-07.57-109068-LabPLUS.image"                                    if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_07_5X_PLUS
 	#
 	default "FRITZ.Box_7590_AX-07.31.image"                                                if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7590_AX-07.57.image"                                                if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7590_AX-07.57-108545-LabPLUS.image"                                 if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+	default "FRITZ.Box_7590_AX-07.70-110409-Labor.image"                                   if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_8X_LABOR
+	default "FRITZ.Box_7590_AX-07.70-110391-Inhaus.image"                                  if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_8X_INHAUS
 	default "FRITZ.Box_7590_AX-07.80.image"                                                if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_8X
 	#
 	default "FRITZ.Smart_Gateway-07.59-Release.image"                                      if FREETZ_TYPE_GATE &&                        FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Smart_Gateway-07.59-110922-Inhaus.image"                                if FREETZ_TYPE_GATE &&                        FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "fw_Speedport_W501V_v_28.04.38.image"                                          if FREETZ_TYPE_W501V
 
@@ -775,7 +802,7 @@
 	string "AVM alien web interface firmware source"                                if FREETZ_DL_OVERRIDE
 	depends on FREETZ_DL_ALIEN
 	#
-	default "FRITZ.Box_WLAN_Repeater_DVB_C.en-de-es-it-fr-pl.133.07.02.image"       if FREETZ_TYPE_1759_1750
+	default "FRITZ.Box_WLAN_Repeater_DVB_C.en-de-es-it-fr-pl.133.07.03.image"       if FREETZ_TYPE_1759_1750
 	#
 	default "fritz.box_wlan_3170.49.04.58.image"                                    if FREETZ_TYPE_3170_7170
 	default "FRITZ.Box_Fon_WLAN_7112.87.04.88.image"                                if FREETZ_TYPE_7112_7170
