--- config/mod/dl-firmware.in
+++ config/mod/dl-firmware.in	2024-06-23 10:04:27.069365562 +0200
@@ -31,7 +31,7 @@
 	#
 	default "@AVM/fritzwlan/fritzrepeater-1200/{other,deutschland}/fritz.os,@AVM/1200"                                 if FREETZ_TYPE_1200
 	#
-	default "@AVM/fritzwlan/fritzrepeater-1200-ax/{other,deutschland}/fritz.os,@AVM/firmware/1200AX"                   if FREETZ_TYPE_1209
+	default "@AVM/fritzwlan/fritzrepeater-1200-ax/{other,deutschland}/fritz.os,@AVM/1200AX"                            if FREETZ_TYPE_1209
 	#
 	default "@AVM/fritzpowerline/fritzpowerline-1240e/{other,deutschland}/fritz.os"                                    if FREETZ_TYPE_1240
 	#
@@ -51,7 +51,7 @@
 	#
 	default "@AVM/fritzwlan/fritzrepeater-3000/{other,deutschland}/fritz.os,@AVM/3000"                                 if FREETZ_TYPE_3000
 	#
-	default "@AVM/fritzwlan/fritzrepeater-3000-ax/{other,deutschland}/fritz.os"                                        if FREETZ_TYPE_3009
+	default "@AVM/fritzwlan/fritzrepeater-3000-ax/{other,deutschland}/fritz.os,@AVM/3000AX"                            if FREETZ_TYPE_3009
 	#
 	default "@AVM/fritzbox.sl_wlan/firmware"                                                                           if FREETZ_TYPE_3020
 	#
@@ -108,7 +108,7 @@
 	#
 	default "@AVM/fritzbox-5690-pro/{other,deutschland}/fritz.os,@AVM/5690Pro"                                         if FREETZ_TYPE_5690
 	#
-	default "@AVM/fritzwlan/fritzrepeater-6000/deutschland/fritz.os"                                                   if FREETZ_TYPE_6000
+	default "@AVM/fritzwlan/fritzrepeater-6000/deutschland/fritz.os,@AVM/6000"                                         if FREETZ_TYPE_6000
 	#
 	default "@AVM/firmware/6430/65797387"                                                                              if FREETZ_TYPE_6430 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "@AVM/fritzbox-6430-cable/deutschland/fritz.os"                                                            if FREETZ_TYPE_6430
@@ -217,7 +217,7 @@
 	default "@AVM/{fritzbox-7430/deutschland/fritz.os,fritzbox.7430/firmware/deutsch}"                                 if (FREETZ_TYPE_7430 || FREETZ_TYPE_7412_7430) && FREETZ_TYPE_LANG_DE
 	default "@AVM/{fritzbox-7430/other/fritz.os,fritzbox.7430/firmware/english}"                                       if FREETZ_TYPE_7430 && FREETZ_TYPE_LANG_EN
 	#
-	default "@AVM/{fritzbox-7490/deutschland/fritz.os,fritzbox.7490/firmware/deutsch},@1&1/7490"                       if FREETZ_TYPE_7490 && FREETZ_TYPE_LANG_DE
+	default "@AVM/{fritzbox-7490/deutschland/fritz.os,fritzbox.7490/firmware/deutsch},@1&1/7490,@AVM/7490"             if FREETZ_TYPE_7490 && FREETZ_TYPE_LANG_DE
 	default "@AVM/{fritzbox-7490/{other,belgium}/fritz.os,fritzbox.7490/firmware/{english,belgium}}"                   if FREETZ_TYPE_7490 && FREETZ_TYPE_LANG_EN
 	#
 	default "@AVM/fritzbox-7510/{deutschland,other}/fritz.os,@AVM/7510"                                                if FREETZ_TYPE_7510
@@ -248,7 +248,7 @@
 	#
 	default "@AVM/fritzbox-7583-vdsl/{deutschland,other}/fritz.os,@AVM/7583VDSL"                                       if FREETZ_TYPE_7584
 	#
-	default "@AVM/{fritzbox-7590/deutschland/fritz.os,fritzbox.7590/firmware/deutsch},@1&1/7590"                       if FREETZ_TYPE_7590 && FREETZ_TYPE_LANG_DE
+	default "@AVM/{fritzbox-7590/deutschland/fritz.os,fritzbox.7590/firmware/deutsch},@1&1/7590,@AVM/7590"             if FREETZ_TYPE_7590 && FREETZ_TYPE_LANG_DE
 	default "@AVM/fritzbox-7590/{other,italy,belgium}/fritz.os"                                                        if FREETZ_TYPE_7590 && FREETZ_TYPE_LANG_EN
 	#
 	default "@AVM/fritzbox-7590-ax/{deutschland,other}/fritz.os,@AVM/7590AX"                                           if FREETZ_TYPE_7599
@@ -339,6 +339,7 @@
 	#
 	default "FRITZ.Repeater_3000_AX-07.42.image"                                           if FREETZ_TYPE_3009 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_3000_AX-07.58.image"                                           if FREETZ_TYPE_3009 && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Repeater_3000_AX-07.90-113972-Inhaus.image"                             if FREETZ_TYPE_3009 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "fritz.box_wlan_3030.21.04.34.image"                                           if FREETZ_TYPE_3030
 	#
@@ -391,11 +392,11 @@
 	default "FRITZ.Box_4040-07.29.image"                                                   if FREETZ_TYPE_4040 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_4040-07.59.image"                                                   if FREETZ_TYPE_4040 && FREETZ_TYPE_FIRMWARE_07_5X
 	#
-	default "FRITZ.Box_4050-07.57-110614-Inhaus.image"                                     if FREETZ_TYPE_4050 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
+	default "FRITZ.Box_4050-07.57-113213-Inhaus.image"                                     if FREETZ_TYPE_4050 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_4060-07.30.image"                                                   if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_4060-07.59.image"                                                   if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_07_5X
-	default "FRITZ.Box_4060-07.90-113797-Inhaus.image"                                     if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
+	default "FRITZ.Box_4060-07.90-114082-Inhaus.image"                                     if FREETZ_TYPE_4060 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "fritz.box_fon_5010.annexa.48.04.43.image"                                     if FREETZ_TYPE_5010
 	#
@@ -417,13 +418,15 @@
 	default "FRITZ.Box_5530-07.30.image"                                                   if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_5530_Fiber-07.58.image"                                             if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_5X
 	default "FRITZ.Box_5530_Fiber-07.81.image"                                             if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_07_8X
+	default "FRITZ.Box_5530_Fiber-07.90-114098-Inhaus.image"                               if FREETZ_TYPE_5530 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_5590-07.30.image"                                                   if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_5590_Fiber-07.58-V2.image/FRITZ.Box_5590_Fiber-07.58.image"         if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_5X
 	default "FRITZ.Box_5590_Fiber-07.81.image"                                             if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_07_8X
 	default "FRITZ.Box_5590_Fiber-07.90-114025-LabBETA.image"                              if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_5590_Fiber-07.90-114023-Inhaus.image"                               if FREETZ_TYPE_5590 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
-	default "FRITZ.Box_5690_Pro-07.59-112003-Inhaus.image"                                 if FREETZ_TYPE_5690 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
+	default "FRITZ.Box_5690_Pro-07.61-113770-Inhaus.image"                                 if FREETZ_TYPE_5690 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Repeater_6000-07.31-V2.image/FRITZ.Repeater_6000-07.31.image"           if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Repeater_6000-07.58.image"                                              if FREETZ_TYPE_6000 && FREETZ_TYPE_FIRMWARE_07_5X
@@ -448,17 +451,23 @@
 	default "FRITZ.Box_6591_Cable-07.29.image"                                             if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_6591_Cable-07.57.image"                                             if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_07_5X
 	default "FRITZ.Box_6591_Cable-07.90-114051-Labor.image"                                if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_6591_Cable-07.90-114092-Inhaus.image"                               if FREETZ_TYPE_6591 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_6660_Cable-07.15.image"                                             if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_6660_Cable-07.29.image"                                             if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_6660_Cable-07.57.image"                                             if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_5X
-	default "FRITZ.Box_6660_Cable-07.90-114052-Labor.image"                                if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_6660_Cable-07.57-108563-LabPLUS.image"                              if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+ 	default "FRITZ.Box_6660_Cable-07.90-114052-Labor.image"                                if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_6660_Cable-07.90-113946-Inhaus.image"                               if FREETZ_TYPE_6660 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_6670_Cable-07.63.image"                                             if FREETZ_TYPE_6670 && FREETZ_TYPE_FIRMWARE_07_5X
+#	default "FRITZ.Box_6670_Cable-07.62-113570-Inhaus.image"                               if FREETZ_TYPE_6670 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_6690_Cable-07.29.image"                                             if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_6690_Cable-07.57-V2.image/FRITZ.Box_6690_Cable-07.57.image"         if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_07_5X
-	default "FRITZ.Box_6690_Cable-07.90-114053-Labor.image"                                if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_6690_Cable-07.57-108550-LabPLUS.image"                              if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_07_5X_PLUS
+ 	default "FRITZ.Box_6690_Cable-07.90-114053-Labor.image"                                if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_6690_Cable-07.90-113944-Inhaus.image"                               if FREETZ_TYPE_6690 && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_6810_LTE.108.06.35.image"                                           if FREETZ_TYPE_6810 && FREETZ_TYPE_LANG_DE && FREETZ_TYPE_FIRMWARE_06_2X
 	default "FRITZ.Box_6810_LTE.en-de-es-it-fr.108.06.35.image"                            if FREETZ_TYPE_6810 && FREETZ_TYPE_LANG_EN && FREETZ_TYPE_FIRMWARE_06_2X
@@ -622,13 +631,16 @@
 	#
 	default "FRITZ.Box_7510-07.30.image"                                                   if FREETZ_TYPE_7510                        && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7510-07.57.image"                                                   if FREETZ_TYPE_7510                        && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7510-07.57-113403-Inhaus.image"                                     if FREETZ_TYPE_7510                        && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_7520-07.14.image"                                                   if FREETZ_TYPE_7520 && FREETZ_TYPE_LANG_DE && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_7520-07.29.image"                                                   if FREETZ_TYPE_7520                        && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7520-07.57.image"                                                   if FREETZ_TYPE_7520                        && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7520-07.57-113418-Inhaus.image"                                     if FREETZ_TYPE_7520                        && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_7520_B-07.31.image"                                                 if FREETZ_TYPE_7521                        && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7520_B-07.57.image"                                                 if FREETZ_TYPE_7521                        && FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7520_B-07.57-113400-Inhaus.image"                                   if FREETZ_TYPE_7521                        && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_7530.en-de-es-it-fr-pl-nl.164.07.02.image"                          if (FREETZ_TYPE_7530 || FREETZ_TYPE_7520_7530) && FREETZ_TYPE_LANG_EN && FREETZ_TYPE_FIRMWARE_07_0X
 	default "FRITZ.Box_7530.07.13.int.image/FRITZ.Box_7530-07.13.image"                    if (FREETZ_TYPE_7530 || FREETZ_TYPE_7520_7530) && FREETZ_TYPE_LANG_EN && FREETZ_TYPE_FIRMWARE_07_1X
@@ -637,11 +649,13 @@
 	default "FRITZ.Box_7530-07.29.image"                                                   if (FREETZ_TYPE_7530 || FREETZ_TYPE_7520_7530)                        && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7530-07.59.image"                                                   if (FREETZ_TYPE_7530 || FREETZ_TYPE_7520_7530)                        && FREETZ_TYPE_FIRMWARE_07_5X
 	default "FRITZ.Box_7530-07.90-114018-Labor.image"                                      if (FREETZ_TYPE_7530 || FREETZ_TYPE_7520_7530)                        && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_7530-07.90-114101-Inhaus.image"                                     if (FREETZ_TYPE_7530 || FREETZ_TYPE_7520_7530)                        && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_7530_AX-07.31.image"                                                if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7530_AX-07.57-V2.image/FRITZ.Box_7530_AX-07.57.image"               if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_5X
 	default "FRITZ.Box_7530_AX-07.81.image"                                                if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_07_8X
-	default "FRITZ.Box_7530_AX-07.90-114021-LabBETA.image"                                 if FREETZ_TYPE_7539 && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+ 	default "FRITZ.Box_7530_AX-07.90-114021-LabBETA.image"                                 if FREETZ_TYPE_7539                        && FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_7530_AX-07.90-114104-Inhaus.image"                                  if FREETZ_TYPE_7539                        && FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_7560.en-de-es-it-fr-pl.149.06.83.image"                             if FREETZ_TYPE_7560 && FREETZ_TYPE_LANG_EN && FREETZ_TYPE_FIRMWARE_06_8X
 	default "FRITZ.Box_7560.en-de-es-it-fr-pl.149.07.01.image"                             if FREETZ_TYPE_7560 && FREETZ_TYPE_LANG_EN && FREETZ_TYPE_FIRMWARE_07_0X
@@ -683,16 +697,20 @@
 	default "FRITZ.Box_7590-07.12.image"                                                   if FREETZ_TYPE_7590 && FREETZ_TYPE_LANG_DE && FREETZ_TYPE_FIRMWARE_07_1X
 	default "FRITZ.Box_7590-07.29.image"                                                   if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7590-07.59.image"                                                   if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_07_5X
-	default "FRITZ.Box_7590-07.90-114019-Labor.image"                                    if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+ 	default "FRITZ.Box_7590-07.90-114019-Labor.image"                                      if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_7590-07.90-114107-Inhaus.image"                                     if FREETZ_TYPE_7590 &&                        FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_7590_AX-07.31.image"                                                if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Box_7590_AX-07.57.image"                                                if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_5X
 	default "FRITZ.Box_7590_AX-07.81.image"                                                if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_07_8X
-	default "FRITZ.Box_7590_AX-07.90-114020-Labor.image"                                   if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+ 	default "FRITZ.Box_7590_AX-07.90-114020-Labor.image"                                   if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_08_0X_LABOR
+	default "FRITZ.Box_7590_AX-07.90-114110-Inhaus.image"                                  if FREETZ_TYPE_7599 &&                        FREETZ_TYPE_FIRMWARE_08_0X_INHAUS
 	#
 	default "FRITZ.Box_7690-07.61.image"                                                   if FREETZ_TYPE_7690 &&                        FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Box_7690-07.61-114118-Inhaus.image"                                     if FREETZ_TYPE_7690 &&                        FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Smart_Gateway-07.60-Release.image"                                      if FREETZ_TYPE_GATE &&                        FREETZ_TYPE_FIRMWARE_07_5X
+	default "FRITZ.Smart_Gateway-07.61-113992-Inhaus.image"                                if FREETZ_TYPE_GATE &&                        FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "fw_Speedport_W501V_v_28.04.38.image"                                          if FREETZ_TYPE_W501V
 
