--- config/mod/prefix.in	2024-04-12 13:22:05.399438415 +0200
+++ config/mod/prefix.in	2024-04-12 19:40:18.726142027 +0200
@@ -112,6 +112,7 @@
 config FREETZ_TYPE_PREFIX_LABOR_FIRMWARE
 	string
 	default "_labor"            if FREETZ_TYPE_FIRMWARE_LABOR
+	default "_laborplus"        if FREETZ_TYPE_FIRMWARE_PLUS
 	default "_inhaus"           if FREETZ_TYPE_FIRMWARE_INHAUS
 
 config FREETZ_TYPE_PREFIX_SERIES_SUBDIR
