--- make/pkgs/irssi/irssi.mk	2022-12-02 09:39:13.823173000 +0100
+++ make/pkgs/irssi/irssi.mk	2022-12-02 11:41:34.381149805 +0100
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, 0.8.15)
+$(call PKG_INIT_BIN, 1.4.3)
 $(PKG)_SOURCE:=irssi-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=c15957cd898cc71eeae389b5a71055693b95794667e637da3327553661e4c6de
-$(PKG)_SITE:=http://irssi.org/files
+$(PKG)_HASH:=2cbaca493d4c36d3f1a6a2a0680ebdfe3bdabc202d9b8b99e49f4c83b7021591
+$(PKG)_SITE:=https://codeberg.org/$(pkg)/$(pkg)/releases/download/$($(PKG)_VERSION)
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/src/fe-text/irssi
 $(PKG)_TARGET_BINARY:=$($(PKG)_DEST_DIR)/usr/bin/irssi
--- make/pkgs/irssi/Config.in	2022-12-02 09:39:13.823173000 +0100
+++ make/pkgs/irssi/Config.in	2022-12-02 11:40:18.198712516 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_IRSSI
-	bool "Irssi 0.8.15"
+	bool "Irssi 1.4.3"
 	select FREETZ_LIB_libglib_2
 	select FREETZ_LIB_libgmodule_2
 	select FREETZ_LIB_libncurses
