--- make/pkgs/tor/tor.mk	2022-11-13 06:18:41.317557000 +0100
+++ make/pkgs/tor/tor.mk	2022-11-13 06:38:26.256011721 +0100
@@ -1,6 +1,6 @@
-$(call PKG_INIT_BIN, 0.4.7.11)
+$(call PKG_INIT_BIN, 0.4.7.12)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=cf3cafbeedbdbc5fd1c0540e74d6d10a005eadff929098393815f867e32a136e
+$(PKG)_HASH:=3b5d969712c467851bd028f314343ef15a97ea457191e93ffa97310b05b9e395
 $(PKG)_SITE:=https://www.torproject.org/dist
 ### WEBSITE:=https://www.torproject.org/download/tor/
 ### MANPAGE:=https://trac.torproject.org/projects/tor/wiki/
--- make/pkgs/tor/Config.in	2022-11-13 06:18:41.317557000 +0100
+++ make/pkgs/tor/Config.in	2022-11-13 06:40:34.973949633 +0100
@@ -2,7 +2,7 @@
 	depends on !FREETZ_OPENSSL_VERSION_1_MIN
 
 config FREETZ_PACKAGE_TOR
-	bool "Tor 0.4.7.11"
+	bool "Tor 0.4.7.12"
 	depends on FREETZ_OPENSSL_VERSION_1_MIN
 	select FREETZ_LIB_libm               if !FREETZ_PACKAGE_TOR_STATIC && FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libevent           if !FREETZ_PACKAGE_TOR_STATIC
