--- make/pkgs/xz/xz.mk	2022-12-11 12:16:40.871583000 +0100
+++ make/pkgs/xz/xz.mk	2022-12-14 17:47:05.013414491 +0100
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, 5.2.9)
+$(call PKG_INIT_BIN, 5.4.0)
 $(PKG)_LIB_VERSION:=$($(PKG)_VERSION)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=287ef163e7e57561e9de590b2a9037457af24f03a46bbd12bf84f3263679e8d2
+$(PKG)_HASH:=5f260e3b43f75cf43ca43d107dd18209f7d516782956a74ddd53288e02a83a31
 $(PKG)_SITE:=https://tukaani.org/xz
 ### WEBSITE:=https://tukaani.org/xz/
 ### CHANGES:=https://git.tukaani.org/?p=xz.git;a=blob_plain;f=NEWS;hb=HEAD
--- make/pkgs/xz/Config.in	2022-12-11 12:16:40.871583000 +0100
+++ make/pkgs/xz/Config.in	2022-12-14 17:49:56.377077668 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_XZ
-	bool "XZ Utils 5.2.9 (binary only)"
+	bool "XZ Utils 5.4.0 (binary only)"
 	select FREETZ_LIB_liblzma
 	default n
 	help
--- make/pkgs/xz/patches/020-extra_flags.patch	2022-12-11 12:16:40.871583000 +0100
+++ make/pkgs/xz/patches/020-extra_flags.patch	2022-12-14 17:47:09.125307847 +0100
@@ -1,6 +1,6 @@
 --- src/liblzma/Makefile.in
 +++ src/liblzma/Makefile.in
-@@ -657,7 +657,7 @@
+@@ -695,7 +695,7 @@
  CCASDEPMODE = @CCASDEPMODE@
  CCASFLAGS = @CCASFLAGS@
  CCDEPMODE = @CCDEPMODE@
