--- make/libs/libonig/libonig.mk	2022-10-02 09:28:13.079561000 +0200
+++ make/libs/libonig/libonig.mk	2022-10-02 15:26:24.028059312 +0200
@@ -1,5 +1,5 @@
-$(call PKG_INIT_LIB, v6.7.1)
-$(PKG)_LIB_VERSION:=4.0.0
+$(call PKG_INIT_LIB, v6.9.8)
+$(PKG)_LIB_VERSION:=5.3.0
 $(PKG)_SOURCE:=onig-$($(PKG)_VERSION).tar.xz
 $(PKG)_HASH:=X
 $(PKG)_SITE:=git@https://github.com/kkos/oniguruma
--- make/libs/libonig/external.in	2022-10-02 09:28:13.079561000 +0200
+++ make/libs/libonig/external.in	2022-10-02 15:27:53.603089852 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libonig.so.4.0.0
+		 ${FREETZ_LIBRARY_DIR}/libonig.so.5.3.0
 
--- make/libs/libonig/external.files	2022-10-02 09:28:13.079561000 +0200
+++ make/libs/libonig/external.files	2022-10-02 15:27:29.879346317 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libonig" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libonig.so.4.0.0"
+[ "$EXTERNAL_FREETZ_LIB_libonig" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libonig.so.5.3.0"
--- make/libs/libonig/patches/100-only_src.patch	2022-10-02 09:28:13.079561000 +0200
+++ make/libs/libonig/patches/100-only_src.patch	2022-10-02 15:22:29.582613667 +0200
@@ -1,11 +1,11 @@
 --- Makefile.am
 +++ Makefile.am
 @@ -2,7 +2,7 @@
-
+ 
  ACLOCAL_AMFLAGS = -I m4
-
+ 
 -SUBDIRS = src test sample
 +SUBDIRS = src
-
+ 
  EXTRA_DIST = oniguruma.pc.in HISTORY README_japanese README.md \
-	index.html index_ja.html \
+ 	index.html index_ja.html make_win.bat \

