--- make/libs/yaml/yaml.mk	2022-10-11 21:19:39.740180000 +0200
+++ make/libs/yaml/yaml.mk	2022-10-17 15:36:41.853925677 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_LIB, 0.1.4)
-$(PKG)_LIB_VERSION:=2.0.2
+$(call PKG_INIT_LIB, 0.2.5)
+$(PKG)_LIB_VERSION:=2.0.9
 $(PKG)_SOURCE:=yaml-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=7bf81554ae5ab2d9b6977da398ea789722e0db75b86bffdaeb4e66d961de6a37
+$(PKG)_HASH:=c642ae9b75fee120b2d96c712538bd2cf283228d2337df2cf2988e3c02678ef4
 $(PKG)_SITE:=http://pyyaml.org/download/libyaml/
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/src/.libs/libyaml-0.so.$($(PKG)_LIB_VERSION)
--- make/libs/yaml/external.in	2022-10-11 21:19:39.740180000 +0200
+++ make/libs/yaml/external.in	2022-10-17 15:42:49.079209602 +0200
@@ -4,5 +4,5 @@
 	default n
 	help
 		externals the following file(s):
-		 ${FREETZ_LIBRARY_DIR}/libyaml-0.so.2.0.2
+		 ${FREETZ_LIBRARY_DIR}/libyaml-0.so.2.0.9
 
--- make/libs/yaml/external.files	2022-10-11 21:19:39.740180000 +0200
+++ make/libs/yaml/external.files	2022-10-17 15:42:42.071341251 +0200
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_LIB_libyaml" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libyaml-0.so.2.0.2"
+[ "$EXTERNAL_FREETZ_LIB_libyaml" == "y" ] && EXTERNAL_FILES+=" ${FREETZ_LIBRARY_DIR}/libyaml-0.so.2.0.9"
