--- config/mod/dl-firmware.in	2022-10-10 19:46:55.710515000 +0200
+++ config/mod/dl-firmware.in	2022-10-11 20:32:21.686465207 +0200
@@ -300,7 +300,7 @@
 	default "FRITZ.Powerline_1260E-07.21-Release.image"                                    if FREETZ_TYPE_1260_V1 && FREETZ_TYPE_FIRMWARE_07_2X
 	default "FRITZ.Powerline_1260E.157.07.29.image"                                        if FREETZ_TYPE_1260_V1 && FREETZ_TYPE_FIRMWARE_07_25
 	#
-	default "FRITZ.Powerline_1260.249.07.29-Release.image"                                 if FREETZ_TYPE_1260_V2 && FREETZ_TYPE_FIRMWARE_07_25
+	default "FRITZ.Powerline_1260.249.07.29.image"		                               if FREETZ_TYPE_1260_V2 && FREETZ_TYPE_FIRMWARE_07_25
 	default "FRITZ.Powerline_1260-07.39-101040-Inhaus.image"                               if FREETZ_TYPE_1260_V2 && FREETZ_TYPE_FIRMWARE_07_5X_INHAUS
 	#
 	default "FRITZ.Box_WLAN_Repeater_1750E.134.06.32.image"                                if (FREETZ_TYPE_1750 || FREETZ_TYPE_1759_1750) && FREETZ_TYPE_FIRMWARE_06_2X
