--- make/pkgs/bind/bind.mk	2022-10-27 06:08:55.535234000 +0200
+++ make/pkgs/bind/bind.mk	2022-11-01 11:48:43.697016244 +0100
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, $(if $(FREETZ_PACKAGE_BIND_VERSION_ABANDON),9.11.37,9.16.33))
+$(call PKG_INIT_BIN, $(if $(FREETZ_PACKAGE_BIND_VERSION_ABANDON),9.11.37,9.16.34))
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.$(if $(FREETZ_PACKAGE_BIND_VERSION_ABANDON),gz,xz)
 $(PKG)_HASH_ABANDON:=0d8efbe7ec166ada90e46add4267b7e7c934790cba9bd5af6b8380a4fbfb5aff
-$(PKG)_HASH_CURRENT:=ec4fbea4b2e368d1824971509e33fa159224ad14b436034c6bcd46104c328d91
+$(PKG)_HASH_CURRENT:=513d2efcf5d526c14a0b33af698b6df17d21cd2e011a43f4a94a4f0c84472f16
 $(PKG)_HASH:=$($(PKG)_HASH_$(if $(FREETZ_PACKAGE_BIND_VERSION_ABANDON),ABANDON,CURRENT))
 $(PKG)_SITE:=https://downloads.isc.org/isc/bind9/$($(PKG)_VERSION),http://ftp.isc.org/isc/bind9/$($(PKG)_VERSION)
 ### WEBSITE:=https://www.isc.org/bind/
--- make/pkgs/bind/Config.in	2022-10-27 06:08:55.535234000 +0200
+++ make/pkgs/bind/Config.in	2022-11-01 12:00:28.946609568 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_BIND
-	bool "BIND 9.11.36/9.16.33"
+	bool "BIND 9.11.36/9.16.34"
 	select FREETZ_LIB_libpthread if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libcrypto          if !FREETZ_PACKAGE_BIND_VERSION_ABANDON
 	select FREETZ_LIB_libcrypto_WITH_EC  if !FREETZ_PACKAGE_BIND_VERSION_ABANDON
