--- make/pkgs/bittwist/bittwist.mk	2022-10-27 06:08:55.535234000 +0200
+++ make/pkgs/bittwist/bittwist.mk	2022-11-01 18:02:32.004919923 +0100
@@ -1,9 +1,10 @@
-$(call PKG_INIT_BIN, 1.1)
+$(call PKG_INIT_BIN, 2.0)
 $(PKG)_SOURCE:=$(pkg)-linux-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=3f42f057845c1d1a9c9d10e4e50cf89666f13cdace932f064b25ef47235a8c6e
+$(PKG)_HASH:=7b98123d2bcb9bec59dc5e610c0c5b5749ef2ed83418956c3ddef3cf5824ea45
+### WEBSITE:=https://sourceforge.net/projects/bittwist/files/Linux/
 $(PKG)_SITE:=@SF/$(pkg)
 
-$(PKG)_BINARIES:=bittwist bittwiste bittwistb
+$(PKG)_BINARIES:=bittwist bittwiste
 $(PKG)_BINARIES_BUILD_DIR:=$($(PKG)_BINARIES:%=$($(PKG)_DIR)/src/%)
 $(PKG)_BINARIES_TARGET_DIR:=$($(PKG)_BINARIES:%=$($(PKG)_DEST_DIR)/usr/bin/%)
 
--- make/pkgs/bittwist/Config.in	2022-10-27 06:08:55.535234000 +0200
+++ make/pkgs/bittwist/Config.in	2022-11-01 18:08:07.442286606 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_BITTWIST
-	bool "bittwist 1.1 (binary only)"
+	bool "bittwist 2.0 (binary only)"
 	select FREETZ_LIB_libpcap
 	default n
 	help
--- make/pkgs/bittwist/external.files	2022-10-27 06:08:55.535234000 +0200
+++ make/pkgs/bittwist/external.files	2022-11-01 18:08:54.097362704 +0100
@@ -1 +1 @@
-[ "$EXTERNAL_FREETZ_PACKAGE_BITTWIST" == "y" ] && EXTERNAL_FILES+=" /usr/bin/bittwist /usr/bin/bittwiste /usr/bin/bittwistb"
+[ "$EXTERNAL_FREETZ_PACKAGE_BITTWIST" == "y" ] && EXTERNAL_FILES+=" /usr/bin/bittwist /usr/bin/bittwiste"
--- make/pkgs/bittwist/external.in	2022-10-27 06:08:55.535234000 +0200
+++ make/pkgs/bittwist/external.in	2022-11-01 18:13:39.666893798 +0100
@@ -6,5 +6,4 @@
 		externals the following file(s):
 		 /usr/bin/bittwist
 		 /usr/bin/bittwiste
-		 /usr/bin/bittwistb
 
--- make/pkgs/bittwist/patches/100_ifaddrs_SIOCGIFHWADDR.patch	2022-10-27 06:08:55.535234000 +0200
+++ make/pkgs/bittwist/patches/100_ifaddrs_SIOCGIFHWADDR.patch	2022-11-01 17:55:00.741796990 +0100
@@ -17,4 +17,4 @@
 +#endif
  #include <pcap.h>
  
- #define BITTWIST_VERSION    "1.1"
+ struct pcap_timeval {
