--- make/pkgs/ccid/ccid.mk	2022-11-19 18:06:54.872655000 +0100
+++ make/pkgs/ccid/ccid.mk	2022-11-19 21:17:59.100013430 +0100
@@ -1,6 +1,6 @@
-$(call PKG_INIT_BIN, 1.5.0)
+$(call PKG_INIT_BIN, 1.5.1)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.bz2
-$(PKG)_HASH:=81549b3422469d503996d03a3aed2ef1375b359167f10d66be9e3844e729322e
+$(PKG)_HASH:=e7a78c398ec0d617a4f98bac70d5b64f78689284dd0ae87d4692e2857f117377
 $(PKG)_SITE:=https://ccid.apdu.fr/files
 ### WEBSITE:=https://ccid.apdu.fr/
 ### MANPAGE:=https://salsa.debian.org/rousseau/CCID/blob/master/README.md
--- make/pkgs/ccid/Config.in	2022-11-19 18:06:54.872655000 +0100
+++ make/pkgs/ccid/Config.in	2022-11-19 21:17:46.652262167 +0100
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_CCID
-	bool "CCID 1.5.0"
+	bool "CCID 1.5.1"
 	select FREETZ_LIB_libpthread if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_libusb_1
 	select FREETZ_PACKAGE_PCSC_LITE
