--- make/dtach/dtach.mk	2022-10-03 09:31:31.347349000 +0200
+++ make/dtach/dtach.mk	2022-10-06 17:13:17.112491383 +0200
@@ -1,9 +1,10 @@
-$(call PKG_INIT_BIN,0.8)
+$(call PKG_INIT_BIN,0.9)
 $(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.gz
-$(PKG)_HASH:=16614ebddf8ab2811d3dc0e7f329c7de88929ac6a9632d4cb4aef7fe11b8f2a9
+$(PKG)_HASH:=32e9fd6923c553c443fab4ec9c1f95d83fa47b771e6e1dafb018c567291492f3
 $(PKG)_SITE:=@SF/dtach
 $(PKG)_BINARY:=$($(PKG)_DIR)/dtach
 $(PKG)_TARGET_BINARY:=$($(PKG)_DEST_DIR)/usr/bin/dtach
+### WEBSITE:=https://sourceforge.net/projects/dtach/files/dtach/
 
 $(PKG_SOURCE_DOWNLOAD)
 $(PKG_UNPACKED)
--- make/dtach/Config.in	2022-10-03 09:31:31.347349000 +0200
+++ make/dtach/Config.in	2022-10-06 17:15:39.163028969 +0200
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_DTACH
-	bool "dtach 0.8 (binary only)"
+	bool "dtach 0.9 (binary only)"
 	default n
 	select FREETZ_LIB_libutil if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	help
