--- make/pkgs/minisatip/minisatip.mk	2022-10-25 17:05:21.652062000 +0200
+++ make/pkgs/minisatip/minisatip.mk	2022-10-25 17:47:24.754348740 +0200
@@ -1,7 +1,7 @@
-$(call PKG_INIT_BIN, 1.0.4)
-$(PKG)_SOURCE:=$(pkg)-$($(PKG)_VERSION).tar.xz
-$(PKG)_HASH:=X
-$(PKG)_SITE:=git@https://github.com/catalinii/minisatip
+$(call PKG_INIT_BIN, 1.2.38)
+$(PKG)_SOURCE:=v$($(PKG)_VERSION).tar.gz
+$(PKG)_HASH:=d2746c6c303c7993d7c221c968c9967bdc0742051d623e6b572b74de5ea9f7c0
+$(PKG)_SITE:=https://github.com/catalinii/minisatip/archive/refs/tags/
 
 $(PKG)_BINARY:=$($(PKG)_DIR)/$(pkg)
 $(PKG)_TARGET_BINARY:=$($(PKG)_DEST_DIR)/usr/bin/$(pkg)
--- make/pkgs/minisatip/Config.in	2022-10-25 17:05:21.652062000 +0200
+++ make/pkgs/minisatip/Config.in	2022-10-25 17:48:44.639778435 +0200
@@ -1,5 +1,5 @@
 config FREETZ_PACKAGE_MINISATIP
-	bool "minisatip 1.0.4 (binary only)"
+	bool "minisatip 1.2.38 (binary only)"
 	default n
 	select FREETZ_LIB_libpthread     if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
 	select FREETZ_LIB_librt          if FREETZ_TARGET_UCLIBC_HAS_multiple_libs
